
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://ise-fizkarlsruhe.github.io/odps4mse/ontologies/stato/report/">
      
      
      
      
      <link rel="icon" href="../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.21">
    
    
      
        <title>Report - Ontology Design Patterns for Materials Science and Engineering</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/main.2a3383ac.min.css">
      
        
        <link rel="stylesheet" href="../../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../.." title="Ontology Design Patterns for Materials Science and Engineering" class="md-header__button md-logo" aria-label="Ontology Design Patterns for Materials Science and Engineering" data-md-component="logo">
      &nbsp;
    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Ontology Design Patterns for Materials Science and Engineering
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Report
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/ISE-FIZKarlsruhe/odps4mse" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../.." title="Ontology Design Patterns for Materials Science and Engineering" class="md-nav__button md-logo" aria-label="Ontology Design Patterns for Materials Science and Engineering" data-md-component="logo">
      &nbsp;
    </a>
    Ontology Design Patterns for Materials Science and Engineering
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/ISE-FIZKarlsruhe/odps4mse" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Introduction
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../ProcessODPs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Process ODPs
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Ontologies
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../terms/find/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Find Terms
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



  <h1>Report</h1>

<table>
<thead>
<tr>
<th>Level</th>
<th>Rule Name</th>
<th>Subject</th>
<th>Property</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000006</td>
<td>IAO:0000602</td>
<td>(forall (x y t) (if (and (SpatialRegion x) (continuantPartOfAt y x t)) (SpatialRegion y))) // axiom label in BFO2 CLIF: [036-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000006</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (SpatialRegion x) (Continuant x))) // axiom label in BFO2 CLIF: [035-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000009</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (TwoDimensionalSpatialRegion x) (SpatialRegion x))) // axiom label in BFO2 CLIF: [039-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000016</td>
<td>IAO:0000602</td>
<td>(forall (x t) (if (and (RealizableEntity x) (existsAt x t)) (exists (y) (and (MaterialEntity y) (specificallyDepends x y t))))) // axiom label in BFO2 CLIF: [063-002]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000016</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (Disposition x) (and (RealizableEntity x) (exists (y) (and (MaterialEntity y) (bearerOfAt x y t)))))) // axiom label in BFO2 CLIF: [062-002]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000018</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (ZeroDimensionalSpatialRegion x) (SpatialRegion x))) // axiom label in BFO2 CLIF: [037-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000019</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (Quality x) (SpecificallyDependentContinuant x))) // axiom label in BFO2 CLIF: [055-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000019</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (exists (t) (and (existsAt x t) (Quality x))) (forall (t_1) (if (existsAt x t_1) (Quality x))))) // axiom label in BFO2 CLIF: [105-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000023</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (Role x) (RealizableEntity x))) // axiom label in BFO2 CLIF: [061-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000026</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (OneDimensionalSpatialRegion x) (SpatialRegion x))) // axiom label in BFO2 CLIF: [038-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000028</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (ThreeDimensionalSpatialRegion x) (SpatialRegion x))) // axiom label in BFO2 CLIF: [040-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000031</td>
<td>IAO:0000602</td>
<td>(iff (GenericallyDependentContinuant a) (and (Continuant a) (exists (b t) (genericallyDependsOnAt a b t)))) // axiom label in BFO2 CLIF: [074-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000034</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (Function x) (Disposition x))) // axiom label in BFO2 CLIF: [064-001]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000040</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (MaterialEntity x) (IndependentContinuant x))) // axiom label in BFO2 CLIF: [019-002]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000040</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (and (Entity x) (exists (y t) (and (MaterialEntity y) (continuantPartOfAt x y t)))) (MaterialEntity x))) // axiom label in BFO2 CLIF: [021-002]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>BFO:0000040</td>
<td>IAO:0000602</td>
<td>(forall (x) (if (and (Entity x) (exists (y t) (and (MaterialEntity y) (continuantPartOfAt y x t)))) (MaterialEntity x))) // axiom label in BFO2 CLIF: [020-002]</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>IAO:0000009</td>
<td>IAO:0000232</td>
<td>"9/22/11 BP: changed the rdfs:label for this class from 'label' to 'datum label' to convey that this class is not intended to cover all kinds of labels (stickers, radiolabels, etc.), and not even all kind of textual labels, but rather the kind of labels occuring in a datum.</td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>IAO:0000115</td>
<td>IAO:0000116</td>
<td>"2012-04-05:</td>
</tr>
<tr>
<td>Barry Smith</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The official OBI definition, explaining the meaning of a class or property: 'Shall be Aristotelian, formalized and normalized. Can be augmented with colloquial definitions'  is terrible.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Can you fix to something like:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A statement of necessary and sufficient conditions explaining the meaning of an expression referring to a class or property.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Alan Ruttenberg</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Your proposed definition is a reasonable candidate, except that it is very common that necessary and sufficient conditions are not given. Mostly they are necessary, occasionally they are necessary and sufficient or just sufficient. Often they use terms that are not themselves defined and so they effectively can't be evaluated by those criteria.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>On the specifics of the proposed definition:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>We don't have definitions of 'meaning' or 'expression' or 'property'. For 'reference' in the intended sense I think we use the term 'denotation'. For 'expression', I think we you mean symbol, or identifier. For 'meaning' it differs for class and property. For class we want documentation that let's the intended reader determine whether an entity is instance of the class, or not. For property we want documentation that let's the intended reader determine, given a pair of potential relata, whether the assertion that the relation holds is true. The 'intended reader' part suggests that we also specify who, we expect, would be able to understand the definition, and also generalizes over human and computer reader to include textual and logical definition.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Personally, I am more comfortable weakening definition to documentation, with instructions as to what is desirable.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>We also have the outstanding issue of how to aim different definitions to different audiences. A clinical audience reading chebi wants a different sort of definition documentation/definition from a chemistry trained audience, and similarly there is a need for a definition that is adequate for an ontologist to work with.  @en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>IAO:0000578</td>
<td>IAO:0000116</td>
<td>"Alan, IAO call 20101124: potentially the CRID denotes the instance it was associated with during creation.</td>
</tr>
<tr>
<td>@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>IAO:0000579</td>
<td>IAO:0000112</td>
<td>PubMed is a CRID registry. It has a dataset of PubMed identifiers associated with journal articles. @en</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000070</td>
<td>IAO:0000116</td>
<td>12/3/12: BP: the reference to the 'physical examination' is included to point out that a prediction is not an assay, as that does not require physical examiniation.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000086</td>
<td>IAO:0000232</td>
<td>"Feb 10, 2009.  changes after discussion at OBI Consortium Workshop Feb 2-6, 2009.  accepted as core term.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>May 28 2013. Updated definition taken from ReO based on discussions initiated in Philly 2011 workshop.  Former defnition described a narrower view of reagents in chemistry that restricts bearers of the role to be chemical entities (\""a role played by a molecular entity used to produce a chemical reaction to detect, measure, or produce other substances\"").  Updated definition allows for broader view of reagents in the domain of biomedical research to include larger materials that have parts that participate chemically in a molecular reaction or interaction.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000097</td>
<td>IAO:0000232</td>
<td>"Following OBI call November 2012,26th:</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>1. it was decided there was no need for moving the children class and making them siblings of study subject role.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>2. it also settles the disambiguation about 'study subject'. This is about the individual participating in the investigation/study, Not the 'topic' (as in 'toxicity study') of the investigation/study</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>This note closes the issue and validates the class definition to be part of the OBI core</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>editor = PRS</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000124</td>
<td>IAO:0000112</td>
<td>"The relation between the conclusion \""Gene tpbA is involved in EPS production\"" and the data items produced using two sets of organisms, one being a tpbA knockout, the other being tpbA wildtype tested in polysacharide production assays and analyzed using an ANOVA. "</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000250</td>
<td>IAO:0000115</td>
<td>A molecular label role which inheres in a material entity and which is realized in the process of detecting a molecular dye that imparts color to some material of interest.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000338</td>
<td>IAO:0000112</td>
<td>Concluding that a gene is upregulated in a tissue sample based on the band intensity in a western blot.  Concluding that a patient has a infection based on measurement of an elevated body temperature and reported headache. Concluding that there were problems in an investigation because data from PCR and microarray are conflicting. Concluding that 'defects in gene XYZ cause cancer due to improper DNA repair' based on data from experiments in that study that gene XYZ is involved in DNA repair, and the conclusion of a previous study that cancer patients have an increased number of mutations in this gene.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000339</td>
<td>IAO:0000116</td>
<td>7/18/2011 BP: planning used to itself be a planned process. Barry Smith pointed out that this would lead to an infinite regression, as there would have to be a plan to conduct a planning process, which in itself would be the result of planning etc. Therefore, the restrictions on 'planning' were loosened to allow for informal processes that result in an 'ad hoc plan '. This required changing from 'has_specified_output some plan specifiction' to 'has_participant some plan specification'.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000675</td>
<td>IAO:0000115</td>
<td>is a data transformation objective where the aim is to estimate statistical significance with the aim of proving or disproving a hypothesis by means of some data transformation</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000751</td>
<td>IAO:0000112</td>
<td>In a study in which gene expression is measured in patients between 8 month to 4 years old that have mild or severe malaria and in which the hypothesis is that gene expression in that age group is a function of disease status, the gene expression is the dependent variable.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000938</td>
<td>IAO:0000119</td>
<td>"Bjoern Peters</td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000963</td>
<td>IAO:0000112</td>
<td>The labels 'positive' vs. 'negative', or 'left handed', 'right handed', 'ambidexterous', or 'strongly binding', 'weakly binding' , 'not binding', or '+++', '++', '+', '-' etc. form scales of categorical labels.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000968</td>
<td>IAO:0000116</td>
<td>"2012-12-17 JAO: In common lab usage, there is a distinction made between devices and reagents that is difficult to model. Therefore we have chosen to specifically exclude reagents from the definition of \""device\"", and are enumerating the types of roles that a reagent can perform.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>2013-6-5 MHB: The following clarifications are outcomes of the May 2013 Philly Workshop. Reagents are distinguished from devices that also participate in scientific techniques by the fact that reagents are chemical or biological in nature and necessarily participate in some chemical interaction or reaction during the realization of their experimental role. By contrast, devices do not participate in such chemical reactions/interactions.  Note that there are cases where devices use reagent components during their operation, where the reagent-device distinction is less clear.  For example:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>(1) An HPLC machine is considered a device, but has a column that holds a stationary phase resin as an operational component.  This resin qualifies as a device if it participates purely in size exclusion, but bears a reagent role that is realized in the running of a column if it interacts electrostatically or chemically with the evaluant. The container the resin is in (“the column”) considered alone is a device. So the entire column as well as the entire HPLC machine are devices that have a reagent as an operating part.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>(2) A pH meter is a device, but its electrode component bears a reagent role in virtue of its interacting directly with the evaluant in execution of an assay.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>(3) A gel running  box  is a device that has a metallic lead as a component that participates in a chemical reaction with the running buffer when a charge is passed through it.  This metallic lead is considered to have a reagent role as a component of this device realized in the running of a gel.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>In the examples above, a reagent is an operational component of a device, but the device itself does not realize a reagent role (as bearing a reagent role is not transitive across the part_of relation).  In this way, the asserted disjointness between a reagent and device holds, as both roles are never realized in the same bearer during execution of an assay.  "</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0000973</td>
<td>IAO:0000115</td>
<td>"A measurement datum that representing the primary structure of a macromolecule(it's sequence) sometimes associated with an indicator of confidence of that measurement.</td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001172</td>
<td>IAO:0000115</td>
<td>A data item of paired values, one indicating the dose of a material, the other quantitating a measured effect at that dose. The dosing intervals are chosen so that effect values be interpolated by a plotting a curve.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001265</td>
<td>IAO:0000119</td>
<td>adapted from wikipedia (http://en.wikipedia.org/wiki/Familywise_error_rate) @en</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001404</td>
<td>IAO:0000232</td>
<td>"MO definition:</td>
</tr>
<tr>
<td>The genotype of the individual organism from which the biomaterial was derived. Individual genetic characteristics include polymorphisms, disease alleles, and haplotypes.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>examples in ArrayExpress</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>wild_type</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>MutaMouse (CD2F1 mice with lambda-gt10LacZ integration)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>AlfpCre; SNF5 flox/knockout</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>p53 knock out</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>C57Bl/6 gp130lox/lox MLC2vCRE/+</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>fer-15; fem-1</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>df/df</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>pat1-114/pat1-114 ade6-M210/ade6-M216 h+/h+ (cells are diploid)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001573</td>
<td>IAO:0000112</td>
<td>The part of a FASTA file that contains the letters ACTGGGAA</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001573</td>
<td>IAO:0000232</td>
<td>8/29/11 call: This is added after a request from Melanie and Yu. They should review it further. This should be a child of 'sequence data', and as of the current definition will infer there.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001834</td>
<td>IAO:0000112</td>
<td>"Concluding that the length of the hypotenuse is equal to the square root of the sum of squares of the other two sides in a right-triangle.</td>
</tr>
<tr>
<td>Concluding that a gene is upregulated in a tissue sample based on the band intensity in a western blot.  Concluding that a patient has a infection based on measurement of an elevated body temperature and reported headache. Concluding that there were problems in an investigation because data from PCR and microarray are conflicting.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001909</td>
<td>IAO:0000116</td>
<td>"In the Philly 2013 workshop, we recognized the limitations of \""conclusion textual entity\"", and we introduced this as more general. The need for the 'textual entity' term going forward is up for future debate. "</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>OBI:0001912</td>
<td>IAO:0000115</td>
<td>A processed material that serves as a liquid vehicle for freezing cells for long term quiescent stroage, which contains chemicls needed to sustain cell viability across freeze-thaw cycles.</td>
</tr>
<tr>
<td>WARN</td>
<td>annotation_whitespace</td>
<td>RO:0001901</td>
<td>IAO:0000115</td>
<td>"</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## Elucidation</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>This is used when the statement/axiom is assumed to hold true 'eternally'</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## How to interpret (informal)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>First the \""atemporal\"" FOL is derived from the OWL using the standard</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>interpretation. This axiom is temporalized by embedding the axiom</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>within a for-all-times quantified sentence. The t argument is added to</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>all instantiation predicates and predicates that use this relation.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## Example</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Class: nucleus</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>SubClassOf: part_of some cell</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>forall t :</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>forall n :</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>instance_of(n,Nucleus,t)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>implies</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>exists c :</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>instance_of(c,Cell,t)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>part_of(n,c,t)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## Notes</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>This interpretation is <em>not</em> the same as an at-all-times relation</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>duplicate_exact_synonym</td>
<td>STATO:0000636</td>
<td>IAO:0000118</td>
<td>NNS</td>
</tr>
<tr>
<td>WARN</td>
<td>duplicate_exact_synonym</td>
<td>STATO:0000637</td>
<td>IAO:0000118</td>
<td>NNS</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000070</td>
<td>OBI:0000417</td>
<td>OBI:0000441</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000094</td>
<td>OBI:0000417</td>
<td>OBI:0000456</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000274</td>
<td>OBI:0000417</td>
<td>OBI:0000434</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000435</td>
<td>OBI:0000299</td>
<td>OBI:0001305</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000443</td>
<td>OBI:0000417</td>
<td>OBI:0000437</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000451</td>
<td>OBI:0000312</td>
<td>OBI:0200169</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000648</td>
<td>OBI:0000312</td>
<td>OBI:0200175</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000649</td>
<td>OBI:0000312</td>
<td>OBI:0200184</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000650</td>
<td>OBI:0000417</td>
<td>OBI:0200031</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000652</td>
<td>OBI:0000417</td>
<td>OBI:0000686</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000662</td>
<td>OBI:0000312</td>
<td>OBI:0000668</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000668</td>
<td>OBI:0000417</td>
<td>OBI:0200186</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000673</td>
<td>OBI:0000417</td>
<td>OBI:0000675</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000674</td>
<td>OBI:0000312</td>
<td>OBI:0200181</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000679</td>
<td>OBI:0000312</td>
<td>OBI:0200170</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000838</td>
<td>OBI:0000417</td>
<td>OBI:0000806</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000932</td>
<td>RO:0000085</td>
<td>OBI:0000372</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0000967</td>
<td>RO:0000085</td>
<td>OBI:0000370</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0001032</td>
<td>RO:0000085</td>
<td>OBI:0000367</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0001834</td>
<td>OBI:0000299</td>
<td>IAO:0000144</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0002089</td>
<td>RO:0000085</td>
<td>OBI:0000401</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0200000</td>
<td>OBI:0000417</td>
<td>OBI:0200166</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0200073</td>
<td>OBI:0000299</td>
<td>OBI:0001265</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0200089</td>
<td>OBI:0000417</td>
<td>OBI:0000791</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0200163</td>
<td>OBI:0000299</td>
<td>OBI:0001442</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0200171</td>
<td>OBI:0000417</td>
<td>OBI:0200172</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0200194</td>
<td>OBI:0000417</td>
<td>OBI:0200083</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:0600014</td>
<td>OBI:0000417</td>
<td>OBI:0000639</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:1110108</td>
<td>RO:0000087</td>
<td>OBI:0000319</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>OBI:1110109</td>
<td>RO:0000087</td>
<td>OBI:0000444</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000027</td>
<td>OBI:0000417</td>
<td>STATO:0000121</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000033</td>
<td>OBI:0000312</td>
<td>OBI:0200117</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000085</td>
<td>OBI:0000295</td>
<td>STATO:0000175</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000119</td>
<td>OBI:0000299</td>
<td>STATO:0000144</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000131</td>
<td>OBI:0000417</td>
<td>STATO:0000183</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000133</td>
<td>BFO:0000062</td>
<td>OBI:0200201</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000137</td>
<td>OBI:0000417</td>
<td>STATO:0000226</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000191</td>
<td>OBI:0000417</td>
<td>STATO:0000224</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000202</td>
<td>OBI:0000417</td>
<td>STATO:0000253</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000247</td>
<td>OBI:0000417</td>
<td>STATO:0000173</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000279</td>
<td>OBI:0000417</td>
<td>STATO:0000255</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000337</td>
<td>OBI:0000299</td>
<td>STATO:0000485</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000443</td>
<td>OBI:0000417</td>
<td>STATO:0000439</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000471</td>
<td>STATO:0000403</td>
<td>STATO:0000039</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_class_axiom_no_genus</td>
<td>STATO:0000697</td>
<td>OBI:0000417</td>
<td>STATO:0000173</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_pair</td>
<td>OBI:0000674</td>
<td>owl:equivalentClass</td>
<td>STATO:0000574</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_pair</td>
<td>OBI:0000679</td>
<td>owl:equivalentClass</td>
<td>STATO:0000573</td>
</tr>
<tr>
<td>WARN</td>
<td>equivalent_pair</td>
<td>STATO:0000247</td>
<td>owl:equivalentClass</td>
<td>STATO:0000697</td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>BFO:0000062</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>BFO:0000063</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>BFO:0000141</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000004</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000039</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000114</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000404</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000406</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000407</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>IAO:0000582</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:10239</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:117571</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:2</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:2157</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:2759</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:314146</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:32523</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:32524</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:33154</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:33213</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:40674</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:7742</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>NCBITaxon:9606</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>RO:0001900</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>RO:0002222</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>obo:obi.owl</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:contributor</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:creator</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:date</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:description</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:format</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:source</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc11:subject</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>missing_definition</td>
<td>dc:license</td>
<td>IAO:0000115</td>
<td></td>
</tr>
<tr>
<td>WARN</td>
<td>multiple_equivalent_classes</td>
<td>OBI:0000674</td>
<td>owl:equivalentClass</td>
<td>STATO:0000574</td>
</tr>
<tr>
<td>WARN</td>
<td>multiple_equivalent_classes</td>
<td>OBI:0000674</td>
<td>owl:equivalentClass</td>
<td>blank node</td>
</tr>
<tr>
<td>WARN</td>
<td>multiple_equivalent_classes</td>
<td>OBI:0000679</td>
<td>owl:equivalentClass</td>
<td>STATO:0000573</td>
</tr>
<tr>
<td>WARN</td>
<td>multiple_equivalent_classes</td>
<td>OBI:0000679</td>
<td>owl:equivalentClass</td>
<td>blank node</td>
</tr>
<tr>
<td>WARN</td>
<td>multiple_equivalent_classes</td>
<td>STATO:0000247</td>
<td>owl:equivalentClass</td>
<td>STATO:0000697</td>
</tr>
<tr>
<td>WARN</td>
<td>multiple_equivalent_classes</td>
<td>STATO:0000247</td>
<td>owl:equivalentClass</td>
<td>blank node</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000016</td>
<td>IAO:0000600</td>
<td>b is a disposition means: b is a realizable entity &amp; b’s bearer is some material entity &amp; b is such that if it ceases to exist, then its bearer is physically changed, &amp; b’s realization occurs when and because this bearer is in some special physical circumstances, &amp; this realization occurs in virtue of the bearer’s physical make-up. (axiom label in BFO2 Reference: [062-002])@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000019</td>
<td>IAO:0000600</td>
<td>a quality is a specifically dependent continuant that, in contrast to roles and dispositions, does not require any further process in order to be realized. (axiom label in BFO2 Reference: [055-001])@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000023</td>
<td>IAO:0000600</td>
<td>b is a role means: b is a realizable entity &amp; b exists because there is some single bearer that is in some special physical, social, or institutional set of circumstances in which this bearer does not have to be&amp; b is not such that, if it ceases to exist, then the physical make-up of the bearer is thereby changed. (axiom label in BFO2 Reference: [061-001])@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000050</td>
<td>IAO:0000115</td>
<td>a core relation that holds between a part and its whole@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000051</td>
<td>IAO:0000115</td>
<td>a core relation that holds between a whole and its part@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000054</td>
<td>IAO:0000600</td>
<td>[copied from inverse property 'realizes'] to say that b realizes c at t is to assert that there is some material entity d &amp; b is a process which has participant d at t &amp; c is a disposition or role of which d is bearer_of at t&amp; the type instantiated by b is correlated with the type instantiated by c. (axiom label in BFO2 Reference: [059-003])@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>BFO:0000055</td>
<td>IAO:0000600</td>
<td>to say that b realizes c at t is to assert that there is some material entity d &amp; b is a process which has participant d at t &amp; c is a disposition or role of which d is bearer_of at t&amp; the type instantiated by b is correlated with the type instantiated by c. (axiom label in BFO2 Reference: [059-003])@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000001</td>
<td>IAO:0000115</td>
<td>a directive information entity that specifies what should happen if the trigger condition is fulfilled@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000005</td>
<td>IAO:0000115</td>
<td>a directive information entity that describes an intended process endpoint. When part of a plan specification the concretization is realized in a planned process in which the bearer tries to effect the world so that the process endpoint is achieved.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000007</td>
<td>IAO:0000115</td>
<td>a directive information entity that describes an action the bearer will take@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000027</td>
<td>IAO:0000115</td>
<td>a data item is an information content entity that is intended to be a truthful statement about something (modulo, e.g., measurement precision or other systematic errors) and is constructed/acquired by a method which reliably tends to produce (approximately) truthful statements.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000032</td>
<td>IAO:0000115</td>
<td>a scalar measurement datum is a measurement datum that is composed of two parts, numerals and a unit label.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000055</td>
<td>IAO:0000115</td>
<td>a rule is an executable which guides, defines, restricts actions@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000102</td>
<td>IAO:0000115</td>
<td>data about an ontology part is a data item about a part of an ontology, for example a term@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000119</td>
<td>IAO:0000115</td>
<td>formal citation, e.g. identifier in external database to indicate / attribute source(s) for the definition. Free text indicate / attribute source(s) for the definition. EXAMPLE: Author Name, URI, MeSH Term C04, PUBMED ID, Wiki uri on 31.01.2007@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000121</td>
<td>IAO:0000115</td>
<td>term created to ease viewing/sort terms for development purpose, and will not be included in a release@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000136</td>
<td>IAO:0000115</td>
<td>is_about is a (currently) primitive relation that relates an information artifact to an entity.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000219</td>
<td>IAO:0000115</td>
<td>denotes is a primitive, instance-level, relation obtaining between an information content entity and some portion of reality. Denotation is what happens when someone creates an information content entity E in order to specifically refer to something. The only relation between E and the thing is that E can be used to 'pick out' the thing. This relation connects those two together. Freedictionary.com sense 3: To signify directly; refer to specifically@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000221</td>
<td>IAO:0000115</td>
<td>"m is a quality measurement of q at t when</td>
</tr>
<tr>
<td>q is a quality</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>there is a measurement process p that has specified output m, a measurement datum, that is about q@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000413</td>
<td>IAO:0000115</td>
<td>relates a process to a time-measurement-datum that represents the duration of the process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000417</td>
<td>IAO:0000115</td>
<td>inverse of the relation of is quality measurement of@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000572</td>
<td>IAO:0000115</td>
<td>a planned process in which a document is created or added to by including the specified input in it.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000581</td>
<td>IAO:0000115</td>
<td>relates a time stamped measurement datum to the time measurement datum that denotes the time when the measurement was taken@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>IAO:0000583</td>
<td>IAO:0000115</td>
<td>relates a time stamped measurement datum to the measurement datum that was measured@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000066</td>
<td>IAO:0000115</td>
<td>a planned process that consists of parts: planning, study design execution, documentation and which produce conclusion(s).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000067</td>
<td>IAO:0000115</td>
<td>a role that inheres in a material entity that is realized in an assay in which data is generated about the bearer of the evaluant role@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000079</td>
<td>IAO:0000115</td>
<td>a processed material that provides the needed nourishment for microorganisms or cells grown in vitro.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000112</td>
<td>IAO:0000115</td>
<td>a role borne by a material entity that is gained during a specimen collection process and that can be realized by use of the specimen in an investigation@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000181</td>
<td>IAO:0000115</td>
<td>a population is a collection of individuals from the same taxonomic class living, counted or sampled at a particular site or in a particular area@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000274</td>
<td>IAO:0000115</td>
<td>is a process with the objective to place a material entity bearing the 'material to be added role' into a material bearing the 'target of material addition role'.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000319</td>
<td>IAO:0000115</td>
<td>material to be added role is a protocol participant role realized by a material which is added into a material bearing the target of material addition role in a material addition process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000339</td>
<td>IAO:0000115</td>
<td>a process of creating or modifying a plan specification@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000416</td>
<td>IAO:0000115</td>
<td>cloning insert role is a role which inheres in DNA or RNA and is realized by the process of being inserted into a cloning vector in a cloning process.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000423</td>
<td>IAO:0000115</td>
<td>an extract is a material entity which results from an extraction process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000427</td>
<td>IAO:0000115</td>
<td>"(protein or rna) or has_part (protein or rna) and</td>
</tr>
<tr>
<td>has_function some GO:0003824 (catalytic activity)@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000434</td>
<td>IAO:0000115</td>
<td>is the specification of an objective to add a material into a target material. The adding is asymmetric in the sense that the target material largely retains its identity@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000435</td>
<td>IAO:0000115</td>
<td>"an assay which generates data about a genotype from a specimen of genomic DNA. A variety of</td>
</tr>
<tr>
<td>techniques and instruments can be used to produce information about sequence variation at particular genomic positions.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000437</td>
<td>IAO:0000115</td>
<td>an assay objective to determine the presence or concentration of an analyte in the evaluant@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000441</td>
<td>IAO:0000115</td>
<td>an objective specification to determine a specified type of information about an evaluated entity (the material entity bearing evaluant role)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000444</td>
<td>IAO:0000115</td>
<td>target of material addition role is a role realized by an entity into which a material is added in a material addition process</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000456</td>
<td>IAO:0000115</td>
<td>an objective specifiction that creates an specific output object from input materials.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000471</td>
<td>IAO:0000115</td>
<td>a planned process that carries out a study design</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000639</td>
<td>IAO:0000115</td>
<td>is an objective to transform a material entity into spatially separated components.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000643</td>
<td>IAO:0000115</td>
<td>the relation of the cells in the finger of the skin to the finger, in which an indeterminate number of grains are parts of the whole by virtue of being grains in a collective that is part of the whole, and in which removing one granular part does not nec- essarily damage or diminish the whole. Ontological Whether there is a fixed, or nearly fixed number of parts - e.g. fingers of the hand, chambers of the heart, or wheels of a car - such that there can be a notion of a single one being missing, or whether, by contrast, the number of parts is indeterminate - e.g., cells in the skin of the hand, red cells in blood, or rubber molecules in the tread of the tire of the wheel of the car.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000652</td>
<td>IAO:0000115</td>
<td>is a material processing with the objective to combine two or more material entities as input into a single material entity as output.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000671</td>
<td>IAO:0000115</td>
<td>a material obtained from an organism in order to be a representative of the whole</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000675</td>
<td>IAO:0000115</td>
<td>is a data transformation objective where the aim is to estimate statistical significance with the aim of proving or disproving a hypothesis by means of some data transformation</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000686</td>
<td>IAO:0000115</td>
<td>is an objective to obtain an output material that contains several input materials.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000722</td>
<td>IAO:0000115</td>
<td>is a collection of short paired tags from the two ends of DNA fragments are extracted and covalently linked as ditag constructs</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000736</td>
<td>IAO:0000115</td>
<td>is a collection of short tags from DNA fragments, are extracted and covalently linked as single tag constructs</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000750</td>
<td>IAO:0000115</td>
<td>a directive information entity that is part of a study design. Independent variables are entities whose values are selected to determine its relationship to an observed phenomenon (the dependent variable). In such an experiment, an attempt is made to find evidence that the values of the independent variable determine the values of the dependent variable (that which is being measured). The independent variable can be changed as required, and its values do not represent a problem requiring explanation in an analysis, but are taken simply as given. The dependent variable on the other hand, usually cannot be directly controlled@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000751</td>
<td>IAO:0000115</td>
<td>dependent variable specification is part of a study design. The dependent variable is the event studied and expected to change when the independent variable varies.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000811</td>
<td>IAO:0000115</td>
<td>a quality of a DNA molecule that inheres in its bearer due to the order of its DNA nucleotide residues.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000838</td>
<td>IAO:0000115</td>
<td>a process with that achieves the objective to maintain some or all of the characteristics of an input material over time</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0000931</td>
<td>IAO:0000115</td>
<td>the part of the execution of an intervention design study which is varied between two or more subjects in the study</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001032</td>
<td>IAO:0000115</td>
<td>a device which has a function to emit light.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001143</td>
<td>IAO:0000115</td>
<td>a labeled specimen that is the output of a labeling process and has grain labeled nucleic acid for detection of the nucleic acid in future experiments.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001225</td>
<td>IAO:0000115</td>
<td>a genetic characteristics information which is a part of genotype information that identifies the population of organisms@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001305</td>
<td>IAO:0000115</td>
<td>a genetic characteristics information that is about the genetic material of an organism and minimally includes information about the genetic background and can in addition contain information about specific alleles, genetic modifications, etc.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001352</td>
<td>IAO:0000115</td>
<td>a genetic alteration information that about one of two or more alternative forms of a gene or marker sequence and differing from other alleles at one or more mutational sites based on sequence. Polymorphisms are included in this definition.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001364</td>
<td>IAO:0000115</td>
<td>a genetic characteristics information that is about known changes or the lack thereof from the genetic background, including allele information, duplication, insertion, deletion, etc.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001404</td>
<td>IAO:0000115</td>
<td>a data item that is about genetic material including polymorphisms, disease alleles, and haplotypes.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0001936</td>
<td>IAO:0000115</td>
<td>a material entity that is the specified output of an addition of molecular label process that aims to label some molecular target to allow for its detection in a detection of molecular label assay</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0100064</td>
<td>IAO:0000115</td>
<td>a screening library is a collection of materials engineered to identify qualities of a subset of its members during a screening process?@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0200044</td>
<td>IAO:0000115</td>
<td>an agglomerative hierarchical clustering which generates successive clusters based on a distance measure, where the distance between two clusters is calculated as the maximum distance between objects from the first cluster and objects from the second cluster.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0200066</td>
<td>IAO:0000115</td>
<td>a data transformation that performs more than one hypothesis test simultaneously, a closed-test procedure, that  controls the familywise error rate for all the k hypotheses at level α in the strong sense. Objective: multiple testing correction</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0300311</td>
<td>IAO:0000115</td>
<td>observation design is a study design in which subjects are monitored in the absence of any active intervention by experimentalists.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0302903</td>
<td>IAO:0000115</td>
<td>a planned process by which totally or partially complementary, single-stranded nucleic acids are  combined into a single molecule called heteroduplex or homoduplex to an extent depending on the amount of complementarity.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0500002</td>
<td>IAO:0000115</td>
<td>a study design which use the same individuals and exposure them to a set of conditions. The effect of order and practice can be confounding factor in such designs@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0500003</td>
<td>IAO:0000115</td>
<td>a repeated measure design which ensures that experimental units receive, in sequence, the treatment (or the control), and then, after a specified time interval (aka <em>wash-out periods</em>), switch to the control (or treatment). In this design, subjects (patients in human context) serve as their own controls, and randomization may be used to determine the ordering which a subject receives the treatment and control@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0500014</td>
<td>IAO:0000115</td>
<td>factorial design is_a study design which is used to evaluate two or more factors simultaneously. The treatments are combinations of levels of the factors. The advantages of factorial designs over one-factor-at-a-time experiments is that they are more efficient and they allow interactions to be detected. In statistics, a factorial design experiment is an experiment whose design consists of two or more factors, each with discrete possible values or levels, and whose experimental units take on all possible combinations of these levels across all such factors. Such an experiment allows studying the effect of each factor on the response variable, as well as the effects of interactions between factors on the response variable.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0500015</td>
<td>IAO:0000115</td>
<td>a factorial design which has 2 experimental factors (aka independent variables) and 2 factor levels per experimental factors@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600005</td>
<td>IAO:0000115</td>
<td>a process with the objective to obtain a material entity that was part of an organism for potential future use in an investigation@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600014</td>
<td>IAO:0000115</td>
<td>a material processing in which components of an input material become segregated in space@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600015</td>
<td>IAO:0000115</td>
<td>group assignment is a process which has an organism as specified input and during which a role is assigned@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600024</td>
<td>IAO:0000115</td>
<td>a protocol application in which cells are kept alive in a defined environment outside of an organism.  part of cell_culturing@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600036</td>
<td>IAO:0000115</td>
<td>a process through which a new type of cell culture or cell line is created, either through the isolation and culture of one or more cells from a fresh source, or the deliberate experimental modification of an existing cell culture (e.g passaging a primary culture to become a secondary culture or line, or the immortalization or stable genetic modification of an existing culture or line).</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600038</td>
<td>IAO:0000115</td>
<td>a material processing technique intended to add a molecular label to some input material entity, to allow detection of the molecular target of this label in a detection of molecular label assay@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600047</td>
<td>IAO:0000115</td>
<td>the use of a chemical or biochemical means to infer the sequence of a biomaterial@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0600064</td>
<td>IAO:0000115</td>
<td>a planned process with the objective to insert genetic material into a cloning vector for future replication of the inserted material@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:0666667</td>
<td>IAO:0000115</td>
<td>a material separation to recover the nucleic acid fraction of an input material@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:1000029</td>
<td>IAO:0000115</td>
<td>a phage display library is a collection of materials in which a mixture of genes or gene fragments is expressed and can be individually selected and amplified.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>OBI:1110108</td>
<td>IAO:0000115</td>
<td>a material that is added to another one in a material combination process</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>obo:REO_0000171</td>
<td>IAO:0000115</td>
<td>a reagent role inhering in a molecular entity intended to associate with some molecular target to serve as a proxy for the presence, abundance, or location of this target in a detection of molecular label assay.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>obo:REO_0000280</td>
<td>IAO:0000115</td>
<td>a molecular reagent intended to associate with some molecular target to serve as a proxy for the presence, abundance, or location of this target in a detection of molecular label assay</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000052</td>
<td>IAO:0000115</td>
<td>a relation between a specifically dependent continuant (the dependent) and an independent continuant (the bearer), in which the dependent specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000053</td>
<td>IAO:0000115</td>
<td>a relation between an independent continuant (the bearer) and a specifically dependent continuant (the dependent), in which the dependent specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000056</td>
<td>IAO:0000115</td>
<td>a relation between a continuant and a process, in which the continuant is somehow involved in the process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000057</td>
<td>IAO:0000115</td>
<td>a relation between a process and a continuant, in which the continuant is somehow involved in the process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000079</td>
<td>IAO:0000115</td>
<td>a relation between a function and an independent continuant (the bearer), in which the function specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000080</td>
<td>IAO:0000115</td>
<td>a relation between a quality and an independent continuant (the bearer), in which the quality specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000081</td>
<td>IAO:0000115</td>
<td>a relation between a role and an independent continuant (the bearer), in which the role specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000085</td>
<td>IAO:0000115</td>
<td>a relation between an independent continuant (the bearer) and a function, in which the function specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000086</td>
<td>IAO:0000115</td>
<td>a relation between an independent continuant (the bearer) and a quality, in which the quality specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0000087</td>
<td>IAO:0000115</td>
<td>a relation between an independent continuant (the bearer) and a role, in which the role specifically depends on the bearer for its existence@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0001000</td>
<td>IAO:0000115</td>
<td>a relation between two distinct material entities, the new entity and the old entity, in which the new entity begins to exist when the old entity ceases to exist, and the new entity inherits the significant portion of the matter of the old entity@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0001001</td>
<td>IAO:0000115</td>
<td>a relation between two distinct material entities, the old entity and the new entity, in which the new entity begins to exist when the old entity ceases to exist, and the new entity inherits the significant portion of the matter of the old entity@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0001015</td>
<td>IAO:0000115</td>
<td>a relation between two independent continuants, the location and the target, in which the target is entirely within the location@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0001025</td>
<td>IAO:0000115</td>
<td>a relation between two independent continuants, the target and the location, in which the target is entirely within the location@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>RO:0001901</td>
<td>IAO:0000115</td>
<td>"</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## Elucidation</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>This is used when the statement/axiom is assumed to hold true 'eternally'</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## How to interpret (informal)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>First the \""atemporal\"" FOL is derived from the OWL using the standard</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>interpretation. This axiom is temporalized by embedding the axiom</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>within a for-all-times quantified sentence. The t argument is added to</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>all instantiation predicates and predicates that use this relation.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## Example</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Class: nucleus</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>SubClassOf: part_of some cell</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>forall t :</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>forall n :</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>instance_of(n,Nucleus,t)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>implies</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>exists c :</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>instance_of(c,Cell,t)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>part_of(n,c,t)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>## Notes</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>This interpretation is <em>not</em> the same as an at-all-times relation</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000001</td>
<td>IAO:0000115</td>
<td>property to indicate that a design declares a variable; the inverse property is 'is declared by'@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000002</td>
<td>IAO:0000115</td>
<td>an electronic file is an information content entity which conforms to a specification or format and which is meant to hold data and information in digital form, accessible to software agents@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000003</td>
<td>IAO:0000115</td>
<td>a balanced design is a an experimental design where all experimental group have the an equal number of subject observations@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000004</td>
<td>IAO:0000115</td>
<td>property to indicate the variables declared by a design; the inverse property is 'declares'@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000005</td>
<td>IAO:0000115</td>
<td>a single factor design is a study design which declares exactly 1 independent variable@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000006</td>
<td>IAO:0000115</td>
<td>x-axis is a cartesian coordinate axis which is orthogonal to the y-axis and the z-axis@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000007</td>
<td>IAO:0000115</td>
<td>an axis is a line graph used as reference line for the measurement of coordinates.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000008</td>
<td>IAO:0000115</td>
<td>y-axis is a cartesian coordinate axis which is orthogonal to the x-axis and the z-axis@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000011</td>
<td>IAO:0000115</td>
<td>a cartesian axis is one of 3 the axis in a cartesian coordinate system defining a referential in 3 dimensions. each of the axis is orthogonal to the other 2@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000012</td>
<td>IAO:0000115</td>
<td>z-axis is a cartesian coordinate axis which is orthogonal to the x-axis and the y-axis@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000013</td>
<td>IAO:0000115</td>
<td>a 2 dimensional cartesian coordinate system is a cartesian coordinate system which defines 2 orthogonal one dimensional axes and which may be used to describe a 2 dimensional spatial region.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000019</td>
<td>IAO:0000115</td>
<td>normal distribution hypothesis is a goodness of fit hypothesis stating that the distribution computed from the sample population fits a normal distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000021</td>
<td>IAO:0000115</td>
<td>a confidence interval which covers 90% of the sampling distribution, meaning that there is a 90% risk of false positive (type I error)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000024</td>
<td>IAO:0000115</td>
<td>a three dimensional cartesian coordinate system is a cartesian coordinate system which defines 3 orthogonal one dimensional axes and which may be used to describe a 3 dimensional spatial region.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000027</td>
<td>IAO:0000115</td>
<td>linkage between 2 categorical variable test is a statistical test which evaluates if there is an association between a predictor variable assuming discrete values and a response variable also assuming discrete values@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000028</td>
<td>IAO:0000115</td>
<td>measure of variation or statistical dispersion is a data item which describes how much a theoritical distribution or dataset is spread.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000029</td>
<td>IAO:0000115</td>
<td>a measure of central tendency is a data item which attempts to describe a set of data by identifying the value of its centre.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000031</td>
<td>IAO:0000115</td>
<td>binary classification (or binomial classification) is a data transformation which aims to cast members of a set into 2 disjoint groups depending on whether the element have a given property/feature or not.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000032</td>
<td>IAO:0000115</td>
<td>an alternative term used for STATO statistical ontology and ISA team@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000034</td>
<td>IAO:0000115</td>
<td>a model parameter is a data item which is part of a model and which is meant to characterize an theoritecal or unknown population. a model parameter may be estimated by considering the properties of samples presumably taken from the theoritecal population@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000035</td>
<td>IAO:0000115</td>
<td>the range is a measure of variation which describes the difference between the lowest score and the highest score in a set of numbers (a data set)</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000038</td>
<td>IAO:0000115</td>
<td>a set of 2 subjects which result from a pairing process which assigns subject to a set based on a pairing rule/criteria@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000039</td>
<td>IAO:0000115</td>
<td>a statistic is a measurement datum to describe a dataset or a variable. It is generated by a calculation on set of observed data.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000040</td>
<td>IAO:0000115</td>
<td>an MA plot is a scatter plot of the log intensity ratios M = log_2(T/R) versus the average log intensities A = log_2(T*T)/2, where T and R represent the signal intensities in the test and reference channels respectively.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000041</td>
<td>IAO:0000115</td>
<td>a R command syntax or link to a R documentation in support of Statistical Ontology Classes or Data Transformations@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000043</td>
<td>IAO:0000115</td>
<td>a false positive rate whose value is 5 per cent@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000044</td>
<td>IAO:0000115</td>
<td>one-way anova is an analysis of variance where the different groups being compared are associated with the factor levels of only one independent variable. The null hypothesis is an absence of difference between the means calculated for each of the groups. The test assumes normality and equivariance of the data.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000045</td>
<td>IAO:0000115</td>
<td>two-way anova is an analysis of variance where the different groups being compared are associated the factor levels of exatly 2 independent variables. The null hypothesis is an absence of difference between the means calculated for each of the groups. The test assumes normality and equivariance of the data.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000046</td>
<td>IAO:0000115</td>
<td>a block design is a kind of study design which declares a blocking variable (also known as nuisance variable) in order to account for a known source of variation and reduce its impact on the acquisition of the signal@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000047</td>
<td>IAO:0000115</td>
<td>a count is a data item denoted by an integer and representing the number of instances or occurences of an entity@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000050</td>
<td>IAO:0000115</td>
<td>signal to noise ratio is a measurement datum comparing the amount of meaningful, useful or interesting data (the signal) to the amount of irrelevant or false data (the noise). Depending on the field and domain of application, different variables will be used to determinate a 'signal to noise ratio'. In statistics, the definition of signal to noise ratio is the ratio of the mean of a measurement to its standard deviation. It thus corresponds to the inverse of the coefficient of variation@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000053</td>
<td>IAO:0000115</td>
<td>a false positive rate is a data item which accounts for the proportion of incorrect rejection of a true null hypothesis.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000054</td>
<td>IAO:0000115</td>
<td>homoskedasticity states that all variances under consideration are homogenous.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000055</td>
<td>IAO:0000115</td>
<td>chromosome coordinate system is a genomic coordinate which uses chromosome of a particular assembly build process to define start and end positions. This coordinate system is unstable and will change with each new genome sequence assembly build.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000056</td>
<td>IAO:0000115</td>
<td>a null hypothesis which states that no linkage exists between 2 categorical variables@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000058</td>
<td>IAO:0000115</td>
<td>goodness of fit hypothesis is a null hypothesis stating that the distribution computed from the sample population fits a theoretical distribution or that a dataset can be correctly explained by a model@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000059</td>
<td>IAO:0000115</td>
<td>the Student's t distribution is a continuous probability distribution which  arises when estimating the mean of a normally distributed population in situations where the sample size is small and population standard deviation is unknown.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000060</td>
<td>IAO:0000115</td>
<td>hypergeometric distribution is a probability distribution that describes the probability of  k successes in n draws from a finite population of size  N containing  K successes without replacement@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000062</td>
<td>IAO:0000115</td>
<td>is a null hypothesis stating that there are no difference observed across a series of measurements made one same subject.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000063</td>
<td>IAO:0000115</td>
<td>genomic coordinate datum is a data item which denotes a genomic position expressed using a genomic coordinate system@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000064</td>
<td>IAO:0000115</td>
<td>sequence read count is a data item determining how many sequence reads have been generated by a DNA sequencing assay for a given stretch of DNA</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000067</td>
<td>IAO:0000115</td>
<td>a continuous probability distribution is a probability distribution which is defined by a probability density function@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000071</td>
<td>IAO:0000115</td>
<td>reaction rate is a measurement datum which represents the speed of a chemical reaction turning reactive species into product species of event (i.e the number of such conversions)s occuring over a time interval@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000072</td>
<td>IAO:0000115</td>
<td>substrate concentration is a scalar measurement datum which denotes the amount of molecular entity involved in an enzymatic reaction (or catalytic chemical reaction) and whose role in that reaction is as substrate.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000075</td>
<td>IAO:0000115</td>
<td>a rarefaction curve is a graph used for estimating species richness in ecology studies@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000080</td>
<td>IAO:0000115</td>
<td>"the Brown Forsythe test is a statistical test which evaluates if the variance of different groups are equal. It relies on computing the median rather than the mean, as used in the Levene's test for homoschedacity.</td>
</tr>
<tr>
<td>This test maybe used to, for instance, ensure that the conditions of applications of ANOVA are met.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000082</td>
<td>IAO:0000115</td>
<td>a fixed effect model is a statistical model which represents the observed quantities in terms of explanatory variables that are treated as if the quantities were non-random.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000084</td>
<td>IAO:0000115</td>
<td>multinomial logistic regression model is a model which attempts to explain data distribution associated with <em>polychotomous</em> response/dependent variable in terms of values assumed by the independent variable uses a function of predictor/independent variable(s): the function used in this instance of regression modeling is probit function.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000085</td>
<td>IAO:0000115</td>
<td>effect size estimate is a data item about the direction and strength of the consequences of a causative agent as explored by statistical methods. Those methods produce estimates of the effect size, e.g. confidence interval@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000086</td>
<td>IAO:0000115</td>
<td>an F-test is a statistical test which evaluates that the computed test statistics follows an F-distribution under the null hypothesis. The F-test is sensitive to departure from normality. F-test arise when decomposing the variability in a data set in terms of sum of squares.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000087</td>
<td>IAO:0000115</td>
<td>a polychotomous variable  is a categorical variable  which is defined to have minimally 2 categories or possible values@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000088</td>
<td>IAO:0000115</td>
<td>statistical sample size is a count evaluating the number of individual experimental units@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000089</td>
<td>IAO:0000115</td>
<td>a case-control study design is a observation study design which assess the risk of particular outcome (a trait or a disease) associated with an event (either an exposure or endogenous factor). A case-control study design therefore declares an exposure variable which is dichotomous in nature (exposed/non-exposed) and an outcome variable, which is also dichotomous (case or control), thus giving the name to the design. During the execution of the design, a case control study defines a population and counts the events to determine their frequency.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000090</td>
<td>IAO:0000115</td>
<td>a dichotomous variable  is a categorical variable  which is defined to have only 2 categories or possible values@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000095</td>
<td>IAO:0000115</td>
<td>paired t-test is a statistical test which is specifically designed to analysis differences between paired observations in the case of studies realizing repeated measures design with only 2 repeated measurements per subject (before and after treatment for example)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000096</td>
<td>IAO:0000115</td>
<td>stratification is a planned process which executes a stratification rule using as input a population and assign it member to mutually exclusive subpopulation based on the values defined by the stratification rule@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000099</td>
<td>IAO:0000115</td>
<td>a random effect(s) model, also called a variance components model, is a kind of hierarchical linear model. It assumes that the dataset being analysed consists of a hierarchy of different populations whose differences relate to that hierarchy.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000100</td>
<td>IAO:0000115</td>
<td>"standardized mean difference is statistic computed by forming the difference between two means, divided by an estimate of the within-group standard deviation.</td>
</tr>
<tr>
<td>It is used to provide an estimation of the effect size between two treatments when the predictor (independent variable) is categorical and the response(dependent) variable is continuous.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A standardized mean difference is a statistic that is a difference between two means, divided by a statistical measure of dispersion.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The term Standardized Mean Difference is a description of the concept without an explicit type of statistical measure of dispersion. If the statistical measure of dispersion is specified, then a type (child term) of Standardized Mean Difference is preferred.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000101</td>
<td>IAO:0000115</td>
<td>the relationship between a fraction and the number above the line@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000102</td>
<td>IAO:0000115</td>
<td>relationship between a planned process and the plan specification that it carries out; it is defined as equivalent to the composed relationship (realizes o concretizes)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000103</td>
<td>IAO:0000115</td>
<td>the multinomial distribution is a probability distribution which gives the probability of any particular combination of numbers of successes for various categories defined in the context of  n independent trials each of which leads to a success for exactly one of k categories, with each category having a given fixed success probability.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000105</td>
<td>IAO:0000115</td>
<td>log signal intensity ratio is a data item which corresponding the logarithmitic base 2 of the ratio between 2 signal intensity, each corresponding to a condition.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000106</td>
<td>IAO:0000115</td>
<td>probit regression model is a model which attempts to explain data distribution associated with <em>dichotomous</em> response/dependent variable in terms of values assumed by the independent variable uses a function of predictor/independent variable(s): the function used in this instance of regression modeling is the probit function aka the quantile function, i.e., the inverse cumulative distribution function (CDF), associated with the standard normal distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000107</td>
<td>IAO:0000115</td>
<td>a statistical model is an information content entity which is a formalization of relationships between variables in the form of mathematical equations. A statistical model describes how one or more random variables are related to one or more other variables. The model is statistical as the variables are not deterministically but stochastically related.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000108</td>
<td>IAO:0000115</td>
<td>"linear regression model is a model which attempts to explain data distribution associated with response/dependent variable in terms of values assumed by the independent variable uses a linear function or linear combination of the regression parameters and the predictor/independent variable(s).</td>
</tr>
<tr>
<td>linear regression modeling makes a number of assumptions, which includes homoskedasticity (constance of variance)@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000109</td>
<td>IAO:0000115</td>
<td>multinomial logistic regression model is a model which attempts to explain data distribution associated with <em>polychotomous</em> response/dependent variable in terms of values assumed by the independent variable uses a function of predictor/independent variable(s): the function used in this instance of regression modeling is logistic function.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000111</td>
<td>IAO:0000115</td>
<td>a sequence read is a DNA sequence data which is generated by a DNA sequencer@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000112</td>
<td>IAO:0000115</td>
<td>"a Funnel plot is a scatter plot of treatment effect versus a measure of study size and aims to provide a visual aid to detecting bias or systematic heterogeneity. A symmetric inverted funnel shape arises from a ‘well-behaved’ data set, in which publication bias is unlikely. An asymmetric funnel indicates a relationship between treatment effect and study size.</td>
</tr>
<tr>
<td>Known caveats: If high precision studies really are different from low precision studies with respect to effect size (e.g., due to different populations examined) a funnel plot may give a wrong impression of publication bias. The appearance of the funnel plot can change quite dramatically depending on the scale on the y-axis — whether it is the inverse square error or the trial size.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Funnel plot was introduced by Light and Palmer in 1984.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000113</td>
<td>IAO:0000115</td>
<td>variance is a data item about a random variable or probability distribution. it is equivalent to the square of the standard deviation.  It is one of several descriptors of a probability distribution, describing how far the numbers lie from the mean (expected value).The variance is the second moment of a distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000114</td>
<td>IAO:0000115</td>
<td>relationship between an element and a set it belongs to@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000115</td>
<td>IAO:0000115</td>
<td>relationship between a set and one of its elements@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000116</td>
<td>IAO:0000115</td>
<td>"the process of using statistical analysis for interpreting and communicating \""what the data say\"".@en"</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000117</td>
<td>IAO:0000115</td>
<td>a discrete probability distribution is a probability distribution which is defined by a probability mass function where the random variable can only assume a finite number of values or infinitely countable values@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000118</td>
<td>IAO:0000115</td>
<td>ranking is a data transformation which turns a non-ordinal variable into a Ordinal variable by sorting the values of the input variable and replacing their value by their position in the sorting result@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000119</td>
<td>IAO:0000115</td>
<td>model parameter estimation is a data transformation that finds parameter values (the model parameter estimates) most compatible with the data as judged by the model.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000120</td>
<td>IAO:0000115</td>
<td>"beanplot is a plot in which (one or) multiple batches (\""beans\"") are shown. Each bean consists of a density trace, which is mirrored to</td>
</tr>
<tr>
<td>form a polygon shape. Next to that, a one-dimensional scatter plot shows all the individual measurements, like in a stripchart.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The name beanplot stems from green beans. The density shape can be seen as the pod of a green bean, while the scatter plot shows the seeds inside the pod.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000121</td>
<td>IAO:0000115</td>
<td>the objective of a data transformation to evaluate a null hypothesis of absence of linkage between variables.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000122</td>
<td>IAO:0000115</td>
<td>a pedigree chart is a graph which plots parent child relations@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000123</td>
<td>IAO:0000115</td>
<td>r2 is a correlation coefficient which is computed over the frequency of 2 dichotomous variable and is used as a measure of Linkage Disequilibrium and as input data item to the creation of an LD plot@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000124</td>
<td>IAO:0000115</td>
<td>a stratification rule/criteria is a criteria used to determine population strata so that a stratification process implementing the rule can result in any member of the total population being assigned to one and only one stratum@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000126</td>
<td>IAO:0000115</td>
<td>"volcano plot is a kind of scatter plot which graphs the negative log of the p-value (significance) on the y-axis versus log2 of fold-change between 2 conditions on the  x-axis.</td>
</tr>
<tr>
<td>It is a popular method for visualizing differential occurence of variables between 2 conditions.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000127</td>
<td>IAO:0000115</td>
<td>a confidence interval which covers 99% of the sampling distribution, meaning that there is a 1% risk of false positive (type I error)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000130</td>
<td>IAO:0000115</td>
<td>the Breslow-Day test is a statistical test which evaluates if the odds ratios are homogenous across N 2x2 contingency tables, for instance several 2x2 contingency tables associated with different strata of a stratified population when evaluating the relationship between exposure and outcome or associated with the different samples coming from several centres in a multicentric study in clinical trial context.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000131</td>
<td>IAO:0000115</td>
<td>a sphericity test is a null hypothesis statistical testing procedure which posits a null hypothesis of equality of the variances of the differences between levels of the repeated measures factor@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000134</td>
<td>IAO:0000115</td>
<td>specificity is a measurement datum qualifying a binary classification test and is computed by substracting the false positive rate to the integral numeral 1@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000135</td>
<td>IAO:0000115</td>
<td>"strictly standardized mean difference (SSMS) is a standardized mean difference which corresponds to the  ratio of mean to the standard deviation of the difference between two groups.</td>
</tr>
<tr>
<td>SSMD directly measures the magnitude of difference between two groups.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>SSMD is widely used in High Content Screen for hit selection and quality control.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>When the data is preprocessed using log-transformation as normally done in HTS experiments, SSMD is the mean of log fold change divided by the standard deviation of log fold change with respect to a negative reference.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>In other words, SSMD is the average fold change (on the log scale) penalized by the variability of fold change (on the log scale).</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>For quality control, one index for the quality of an HTS assay is the magnitude of difference between a positive control and a negative reference in an assay plate. For hit selection, the size of effects of a compound (i.e., a small molecule or an siRNA) is represented by the magnitude of difference between the compound and a negative reference. SSMD directly measures the magnitude of difference between two groups. Therefore, SSMD can be used for both quality control and hit selection in HTS experiments.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000137</td>
<td>IAO:0000115</td>
<td>an homoskedasticity test is a statistical test aiming at evaluate if the variances from several random samples are similar@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000138</td>
<td>IAO:0000115</td>
<td>a 2x2 contingency table is a contingency table build for 2 dichotomous variables (i.e. 2 categorical variables, each with only 2 possible outcomes). It is the simplest of contingency tables@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000139</td>
<td>IAO:0000115</td>
<td>a subject pairing is a planned process which executes a pairing rule and results in the creation of sets of 2 subjects meeting the pairing criteria@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000140</td>
<td>IAO:0000115</td>
<td>"a contigency table is a data item which displays the (multivariate) frequency distribution of the possible values of categorical variables.</td>
</tr>
<tr>
<td>The first row of the table corresponds to categories of one categorical variable, the first column of the table corresponds to categories of the other categorical variable, the cells corresponding to each combination of categories is filled with the observed occurences in the sample being considered.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The table also contains marginal total (marginal sums) and grand total of the occurences</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The term contingency table was first used by Karl Pearson in \""On the Theory of Contingency and Its Relation to Association and Normal Correlation\"", part of the Drapers' Company Research Memoirs Biometric Series I published in 1904.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000141</td>
<td>IAO:0000115</td>
<td>acute toxicity study is an investigation which use interventions organized according to a factorial design and a parallel group design to observe the effect of use of high dose xenobiotics in animal models or cellular models@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000144</td>
<td>IAO:0000115</td>
<td>a model parameter estimate is a data item which results from a model parameter estimation process and which provides a numerical value about a model parameter.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000145</td>
<td>IAO:0000115</td>
<td>"the geometric distribution is a negative binomial distribution where r is 1.</td>
</tr>
<tr>
<td>It is useful for modeling the runs of consecutive successes (or failures) in repeated independent trials of a system.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The geometric distribution models the number of successes before one failure in an independent succession of tests where each test results in success or failure.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The geometric distribution with prob = p has density</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>p(x) = p (1-p)^x</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>for x = 0, 1, 2, …, 0 &lt; p ≤ 1.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>If an element of x is not integer, the result of dgeom is zero, with a warning.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The quantile is defined as the smallest value x such that F(x) ≥ p, where F is the distribution function.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000146</td>
<td>IAO:0000115</td>
<td>a null hypothesis stating that there are differences observed between group of subjects@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000149</td>
<td>IAO:0000115</td>
<td>binomial logistic regression model is a model which attempts to explain data distribution associated with <em>dichotomous</em> response/dependent variable in terms of values assumed by the independent variable uses a function of predictor/independent variable(s): the function used in this instance of regression modeling is logistic function.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000150</td>
<td>IAO:0000115</td>
<td>a minimum value is a data item which denotes the smallest value found in a dataset or resulting from a calculation.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000151</td>
<td>IAO:0000115</td>
<td>maximum value is a data item which denotes the largest value found in a dataset or resulting from a calculation.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000152</td>
<td>IAO:0000115</td>
<td>a quartile is a quantile which splits data into sections accrued of 25% of data, so the first quartile delineates 25% of the data, the second quartile delineates 50% of the data and the third quartile, 75 % of the data@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000154</td>
<td>IAO:0000115</td>
<td>"a violin plot is a plot combining the features of box plot and kernel density plot. The violin plot is therefore similar to box plot but it incorporated in the display the probability density of the data at different values.</td>
</tr>
<tr>
<td>Typically violin plots will include a marker for the median of the data and a box indicating the interquartile range, as in standard box plots.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000155</td>
<td>IAO:0000115</td>
<td>meta-analysis is a data transformation which uses the effect size estimates from several independent quantitative scientific studies addressing the same question in order to assess finding consistency.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000156</td>
<td>IAO:0000115</td>
<td>the Scheffe test is a data transformation which evaluates all possible contrasts and adjusting the levels significance by accounting for multiple comparison. The test is therefore conservative. Confidence intervals can be constructed for the corresponding linear regression.  It was developped by  American statistician Henry  Scheffe in 1959.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000157</td>
<td>IAO:0000115</td>
<td>the LSD test  is a statistical test for multiple comparisons of treatments by means of least significant difference following an ANOVA analysis</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000158</td>
<td>IAO:0000115</td>
<td>a null hypothesis which states that a linkage exists between 2 categorical variables@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000161</td>
<td>IAO:0000115</td>
<td>variable distribution is data item which denotes the spatial resolution of data point making up a variable. variable distribution may be compared to a known probability distribution using goodness of fit test or plotting a quantile-quantile plot for visual assessment of the fit.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000162</td>
<td>IAO:0000115</td>
<td>the role played by an entity part of study group as defined by an experimental design and realized in a data analysis and data interpretation@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000163</td>
<td>IAO:0000115</td>
<td>trimmed mean or truncated mean is a measure of central tendency which involves the calculation of the mean after discarding given parts of a probability distribution or sample at the high and low end, and typically discarding an equal amount of both@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000165</td>
<td>IAO:0000115</td>
<td>a pie chart is a graph in which a circular graph is divided into sector illustrating numerical proportion, meaning that the arc length of each sector (and consequently its central angle and area), is proportional to the quantity it represents.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000166</td>
<td>IAO:0000115</td>
<td>the bart chart is a graph resulting from plotting rectangular bars with lengths proportional to the values that they represent.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000167</td>
<td>IAO:0000115</td>
<td>the first quartile is a quartile which splits the lower 25 % of the data@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000168</td>
<td>IAO:0000115</td>
<td>a real time quantitative pcr plot is a line graph which plots the signal fluorescence intensity as a function of the number of PCR cycle@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000170</td>
<td>IAO:0000115</td>
<td>the first quartile is a quartile which splits the 75 % of the data@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000173</td>
<td>IAO:0000115</td>
<td>"homogeneity testing objective is the objective of a data transformation to test a null hypothesis that two or more sub-groups of a population share the same distribution of a single categorical variable.</td>
</tr>
<tr>
<td>For example, do people of different countries have the same proportion of smokers to non-smokers@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000175</td>
<td>IAO:0000115</td>
<td>confidence interval calculation is a data transformation which determines a confidence interval for a given statistical parameter@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000176</td>
<td>IAO:0000115</td>
<td>t-statistic is a statistic computed from observations and used to produce a p-value in statistical test when compared to a Student's t distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000177</td>
<td>IAO:0000115</td>
<td>the beta distribution is a continuous probability distributions defined on the interval [0, 1] parametrized by two positive shape parameters, denoted by α and β, that appear as exponents of the random variable and control the shape of the distribution@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000180</td>
<td>IAO:0000115</td>
<td>standard normal distribution is a normal distribution with variance = 1 and mean=0@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000183</td>
<td>IAO:0000115</td>
<td>sphericity testing objective is a statistical objective of a data transformation which aims to test a null hypothesis of sphericity holds.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000185</td>
<td>IAO:0000115</td>
<td>a 2 by n contingency table is a contingency table built for one dichotomous variable (a categorical variable with only 2 outcomes) and one polychotomous variable (a polychomotomous variable with at least 2 outcomes)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000188</td>
<td>IAO:0000115</td>
<td>average log signal intensity is a data time which corresponds to the sum of 2 distinct logarithm base 2 transformed signal intensity, each corresponding to a distinct condition of signal acquisition, divided by 2.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000191</td>
<td>IAO:0000115</td>
<td>a goodness of fit statistical test is a statistical test which aim to evaluate if a sample distribution can be considered equivalent to a theoretical distribution used as input@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000192</td>
<td>IAO:0000115</td>
<td>a cartesian product is a data transformation which operates on a n Sets to produce a set of all possible ordered n-tuples where each element of the tuple comes from a Set</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000193</td>
<td>IAO:0000115</td>
<td>is a population whose individual members realize (may be expressed as) a combination of  inclusion rule values specifications or resulting from a sampling process (e.g. recruitment followed by randomization to group) on which a number of measurements will be carried out, which may be used as input to statistical tests and statistical inference.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000194</td>
<td>IAO:0000115</td>
<td>self explanatory@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000197</td>
<td>IAO:0000115</td>
<td>a genomic coordinate system is a coordinate system to describe position of sequence on a genomic scaffold (assembly of chromosome, contig....)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000198</td>
<td>IAO:0000115</td>
<td>a statistical test which makes no assumption about the underlying data distribution@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000199</td>
<td>IAO:0000115</td>
<td>"the Mauchly's test for sphericity is a statistical test which evaluates if the variance of the differences between all combinations of the groups are equal, a property known as 'sphericity' in the context of repeated measures. It is used for instance prior to repeated measure ANOVA.</td>
</tr>
<tr>
<td>The test works by assessing if a Wishart-distributed covariance matrix (or transformation thereof) is proportional to a given matrix.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000200</td>
<td>IAO:0000115</td>
<td>the statistical test power is data item which is about a statistical test and is obtained by subtracting the false negative rate (type II error rate) to 1. The power of a statistical test is the probability that it will correctly lead to the rejection of a false null hypothesis (Greene 2000). The statistical power is the ability of a test to detect an effect, if the effect actually exists (High 2000).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000202</td>
<td>IAO:0000115</td>
<td>within subject comparison statistical test is a kind of statistical test which evaluates if a change occurs within one experimental unit over time following a treatment or an event@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000203</td>
<td>IAO:0000115</td>
<td>a cohort is a study group population where the members are human beings which meet inclusion criteria and undergo a longitudinal design@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000204</td>
<td>IAO:0000115</td>
<td>the F-distribution is a continuous probability distribution which arises in the testing of whether two observed samples have the same variance.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000207</td>
<td>IAO:0000115</td>
<td>a planned process which etablishes and states the different  hypothesis to be evaluated during a null hypothesis statistical test@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000209</td>
<td>IAO:0000115</td>
<td>area under curve is a measurement datum which corresponds to the surface define by the x-axis and bound by the line graph represented in a 2 dimensional plot resulting from an integration or integrative calculus. The interpretation of this measurement datum depends on the variables plotted in the graph@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000210</td>
<td>IAO:0000115</td>
<td>is a data item formed by dividing the fluorescence intensity obtained in one channel to that obtained in the other channel, typically the case when considering 2-color microarray data when imaging is done for Cy3 and Cy5 dyes.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000211</td>
<td>IAO:0000115</td>
<td>odds ratio homogeneity hypothesis is a null hypothesis stating that all odds ratio are homogenous, that is remain within the same range.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000212</td>
<td>IAO:0000115</td>
<td>a tetrachoric correlation coefficient is a polychoric correlation coefficient for 2 dichotomous variables used as proxy for correlation between 2 continuous latent variables.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000213</td>
<td>IAO:0000115</td>
<td>discretization as a processing converting a continuous variable into a polychotomous variable by concretizing a set of discretization rules@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000214</td>
<td>IAO:0000115</td>
<td>a confidence interval which covers 50% of the sampling distribution, meaning that there is a 50% risk of false positive (type I error)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000215</td>
<td>IAO:0000115</td>
<td>probit regression model is a model which attempts to explain data distribution associated with <em>ordinal</em> response/dependent variable in terms of values assumed by the independent variable uses a function of predictor/independent variable(s): the function used in this instance of regression modeling is the ordered probit function.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000216</td>
<td>IAO:0000115</td>
<td>a stratum population is a population resulting from a population stratification prior to sampling process which aims to produce homogenous subpopulations from an heterogeneous population by applying one or more stratification criteria@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000217</td>
<td>IAO:0000115</td>
<td>a null hypothesis which states that a given matrix is proportional to a Wishart-distributed covariance matrix@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000219</td>
<td>IAO:0000115</td>
<td>a real time pcr standard curve is a line graph which plots the fluorescence intensity signal as a function of the concentration of a sample used as reference  and used to determine relative abundance of test samples@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000220</td>
<td>IAO:0000115</td>
<td>the false negative rate is a data item which denotes the proportion of missed detection of elements known to be meeting the detection criteria@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000221</td>
<td>IAO:0000115</td>
<td>a random variable (or aleatory variable or stochastic variable) in probability and statistics,  is a variable whose value is subject to variations due to chance (i.e. randomness, in a mathematical sense)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000222</td>
<td>IAO:0000115</td>
<td>graeco-latin square design is_a study design which allows in its simpler form controlling 3 levels of nuisance variables (also known as blocking variables). The 3 nuisance factors are divided into a tabular grid with the property that each row and each column receive each treatment exactly once.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000223</td>
<td>IAO:0000115</td>
<td>group assignment based on blocking variable specification is a kind of group assignment process which takes into account the levels assumed by a blocking variable to allocate subjects or experimental units to a treatment group@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000227</td>
<td>IAO:0000115</td>
<td>"a normal distribution is a continuous probability distribution described by a probability distribution function described here:</td>
</tr>
<tr>
<td>http://mathworld.wolfram.com/NormalDistribution.html@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000228</td>
<td>IAO:0000115</td>
<td>ordinal variable is a categorical variable where  the discrete possible values are ordered or  correspond to an implicit ranking@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000230</td>
<td>IAO:0000115</td>
<td>the expected value (or expectation, mathematical expectation, EV, mean, or the first moment) of a random variable is a data item which corresponds to the weighted average of all possible values that this random variable can take on. The weights used in computing this average correspond to the probabilities in case of a discrete random variable, or densities in case of a continuous random variable. From a rigorous theoretical standpoint, the expected value is the integral of the random variable with respect to its probability measure.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000231</td>
<td>IAO:0000115</td>
<td>a confidence interval which covers 95% of the sampling distribution, meaning that there is a 5% risk of false positive (type I error). If the number of observations made is large enough, the sampling distribution can be assumed to be normal, which entails that 95% of the sampling distributions falls within roughly2 (1.96) standard deviations from the mean.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000232</td>
<td>IAO:0000115</td>
<td>number of PCR cycle is a count which enumerates how many iterations of 'annealing, renaturation, amplification,' rounds (or cycles) are performed during a polymerase chain reaction (PCR) or an assay relying on PCR.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000233</td>
<td>IAO:0000115</td>
<td>sensitivity is a measurement datum qualifying a binary classification test and is computed by substracting the false negative rate to the integral numeral 1@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000234</td>
<td>IAO:0000115</td>
<td>a residual is a data item which is the output of an error estimate or model fitting process and which is an observable estimate of the unobservable error@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000236</td>
<td>IAO:0000115</td>
<td>the coefficient of variation is a normalized measure of dispersion of a probability distribution of frequency distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000238</td>
<td>IAO:0000115</td>
<td>high content screening is a kind of investigation which uses a standardized cellular assays to test the effect of substances (RNAi or small molecules) held in libraries on a cellular phenotype. it relies on microscopy imaging and or flow-cytometry, robotic handling to ensure fast and high-throughput.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000239</td>
<td>IAO:0000115</td>
<td>high throughput screening is a kind of investigation which uses a standardized  assays (cell based, enzymatic or chemometric) to test the effect of substances (RNAi or small molecules) held in libraries on a very specific and measureable outcome (e.g fluorence intensity). it relies on robotic handling to ensure fast and high-throughput in assay performance, data acquisition and hit selection.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000242</td>
<td>IAO:0000115</td>
<td>statistical error is an data item denoting the amount by which an observation differs from the expected value, being based on the whole statistical population from which the statistical unit was chosen randomly@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000243</td>
<td>IAO:0000115</td>
<td>a box plot is a graph which plots datasets relying on their quartiles and the interquartile range to create the box and the whiskers.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000244</td>
<td>IAO:0000115</td>
<td>(Rn +) − (Rn −), where Rn + = (emission intensity of reporter dye)/(emission intensity of passive reference dye) in PCR with template and Rn − = (emission intensity of reporter dye)/(emission intensity of passive reference dye) in PCR without template or early cycles of a real-time reaction. Ct = threshold cycle, i.e., cycle at which a statistically significant increase in ΔRn is first detected@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000247</td>
<td>IAO:0000115</td>
<td>odds ratio homogeneity test is a statistical test which aims to evaluate that null the hypothesis of consistency odds ratio accross different strata of population is true or not@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000248</td>
<td>IAO:0000115</td>
<td>a blocking variable is a independent variable which is used in a blocking process part of an experiment with the purpose of maximizing the signal coming from the main variable.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000249</td>
<td>IAO:0000115</td>
<td>a DNA microarray hybridization is an assay relying on nucleic acid hybridization , which uses a DNA microarray device and a nucleic acid as input. It precedes a data acquisition process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000250</td>
<td>IAO:0000115</td>
<td>group comparison objective is a data transformation objective which aims to determine if 2 or more study group differ with respect to the signal of a response variable@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000252</td>
<td>IAO:0000115</td>
<td>a categorical variable  is a variable  which that can only assume a finite number  of value and cast observation in a small number of categories@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000253</td>
<td>IAO:0000115</td>
<td>the objective of a data transformation to test a null hypothesis of absence of difference within subject holds.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000255</td>
<td>IAO:0000115</td>
<td>the objective of a data transformation to test a null hypothesis of absence of difference withing subject holds.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000256</td>
<td>IAO:0000115</td>
<td>a manhattan plot for gwas is a kind of scatter plot used to facilitate presentation of genome-wide association study (GWAS) data. Genomic coordinates are displayed along the X-axis, with the negative logarithm of the association P-value for each single nucleotide polymorphism displayed on the Y-axis.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000258</td>
<td>IAO:0000115</td>
<td>a variable is a data item which can assume any of a set of values, either as determined by an agent or as randomly occuring through observation.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000259</td>
<td>IAO:0000115</td>
<td>the relationship between a fraction and the number below the line (or divisor)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000260</td>
<td>IAO:0000115</td>
<td>"repeated measure ANOVA is a kind of ANOVA specifically developed for non-independent observations as found when repeated measurements on the sample experimental unit.</td>
</tr>
<tr>
<td>repeated measure ANOVA is sensitive to departure from normality (evaluation using Bartlett's test), more so in the case of unbalanced groups (i.e. different sizes of sample populations).</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Departure from sphericity (evaluation  using Mauchly'test)  used to be an issue which is now handled robustly by modern tools such as R's lme4 or nlme, which accommodate dependence assumptions other than sphericity.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000264</td>
<td>IAO:0000115</td>
<td>a factor level combination is one a possible sets of factor levels resulting from the cartesian product of sets of factor and their levels as defined in a factorial design@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000267</td>
<td>IAO:0000115</td>
<td>grouped bar chart is a kind of bar chart which juxtaposes the  discrete values for each of the possible value of a given categorical variable, thus providing  within group comparison. Grouped bar charts are good for comparing between each element in the categories, and comparing elements across categories. However, the grouping can make it harder to tell the difference between the total of each group.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000269</td>
<td>IAO:0000115</td>
<td>polychoric correlation coefficient is a correlation coefficient which is computed over 2 variables to characterise an association by proxy with 2  (latent) variables which are assumed to be continuous and normally distributed.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000270</td>
<td>IAO:0000115</td>
<td>a full factorial design is a factorial design which ensures that all possible factor level combinations are defined and used so all between group differences can be explored@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000271</td>
<td>IAO:0000115</td>
<td>permutation numbering is a data tranformation allowing to count the number of possible permutations of elements in a set of size n, each element occurring exactly once. This number is factorial n.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000274</td>
<td>IAO:0000115</td>
<td>receiver operational characteristics curve is a graphical plot which illustrates the performance of a binary classifier system as its discrimination threshold (aka cut-off point) is varied by plotting  sensitivity vs (1 − specificity)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000277</td>
<td>IAO:0000115</td>
<td>hit selection is a planned process which in screening processes such as high-throughput screening, lead to the identification of perturbing agent which cause the typical signal generated by a standardized assay to significantly differ from the negative control. The selection hitself results from meeting or exceeding selection threshold (for instance 6 sigma from the mean or SSMD value beyond 5 when compared to positive controls or below -5 when compared to negative controls@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000278</td>
<td>IAO:0000115</td>
<td>pairing rule is a rule which specifies the criteria for deciding on how to associated any 2 entities.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000279</td>
<td>IAO:0000115</td>
<td>between group comparison statistical test is a statistical test which aims to detect difference  between the means computing for each of the study group populations@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000281</td>
<td>IAO:0000115</td>
<td>a false positive rate whose value is 1 per cent@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000283</td>
<td>IAO:0000115</td>
<td>negative binomial probability distribution is a discrete probability distribution of the number of successes in a sequence of Bernoulli trials before a specified (non-random) number of failures (denoted r) occur. The negative binomial distribution, also known as the Pascal distribution or Pólya distribution, gives the probability of  r-1 successes and x  failures in x+r-1 trials, and success on the (x+r)th trial.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000285</td>
<td>IAO:0000115</td>
<td>hypergeometric test is a null hypothesis test which evaluates if a random variable follows a hypergeometric distribution. It is a test of goodness of fit to that distribution. The test is suited for situation aimed at assessing cases of sampling from a finite set without replacements. For instance, testing for enrichment or depletion of elements (e.g GO categories, genes)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000286</td>
<td>IAO:0000115</td>
<td>"a one-tailed test is a statistical test which, assuming an unskewed probability distribution, allocates all of the significance level to evaluate only one hypothesis to explain a difference.</td>
</tr>
<tr>
<td>The one-tailed test provides more power to detect an effect in one direction by not testing the effect in the other direction.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>one-tailed test should be preceded by two-tailed test in order to avoid missing out on detecting alternate effect explaining an observed difference.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000287</td>
<td>IAO:0000115</td>
<td>a two tailed test is a statistical test which assess the null hypothesis of absence of difference assuming a symmetric (not skewed) underlying probability distribution by allocating half of the significance level selected to each of the direction of change which could explain a difference (for example, a difference can be an excess or a loss).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000289</td>
<td>IAO:0000115</td>
<td>"a design matrix is an information content entity which denotes a study design. The design matrix is a n by m matrix where n the number of rows, corresponds to the number of observations (4 rows if quadruplicates) and where m, the number of columns corresponds to the number of independent variables. Each element in the matrix correspond to a discretized value representing one of the factor levels for a given factor.</td>
</tr>
<tr>
<td>A design matrix can be used as input to statistical modeling or statistical analysis.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The design matrix contains data on the independent variables (also called explanatory variables) in statistical models which attempt to explain observed data on a response variable (often called a dependent variable) in terms of the explanatory variables. The theory relating to such models makes substantial use of matrix manipulations involving the design matrix: see for example linear regression. A notable feature of the concept of a design matrix is that it is able to represent a number of different experimental designs and statistical models, e.g., ANOVA, ANCOVA, and linear regression@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000291</td>
<td>IAO:0000115</td>
<td>"a quantile is a data item which corresponds to specific elements x in the range of a variate X.</td>
</tr>
<tr>
<td>the k-th n-tile P_k is that value of x, say x_k, which corresponds to a cumulative frequency of Nk/n (Kenney and Keeping 1962). If n=4, the quantity is called a quartile, and if n=100, it is called a percentile.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000292</td>
<td>IAO:0000115</td>
<td>a decile is a quantile where n=10  and which splits data into sections accrued of 10% of data, so the first decile delineates 10% of the data, the second decile delineates 20% of the data and the nineth decile, 90 % of the data@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000293</td>
<td>IAO:0000115</td>
<td>a percentile is a quantile which splits data into sections accrued of 1% of data, so the first percentile delineates 1% of the data, the second quartile delineates 2% of the data and the 99th percentile, 99 % of the data@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000294</td>
<td>IAO:0000115</td>
<td>absence of negative difference hypothesis is a hypothesis which assumes that a difference  significantly less than a threshold does not exist.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000295</td>
<td>IAO:0000115</td>
<td>absence of negative difference hypothesis is a hypothesis which assumes that a difference significantly greater than a threshold does not exist.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000296</td>
<td>IAO:0000115</td>
<td>absence of depletion difference hypothesis is a hypothesis which assumes that the representation of an element significantly greater than a threshold does not exist.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000297</td>
<td>IAO:0000115</td>
<td>absence of depletion difference hypothesis is a hypothesis which assumes that the representation of an element significantly less than a threshold does not exist.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000298</td>
<td>IAO:0000115</td>
<td>a binomial test is a statistical hypothesis test which evaluates if the observations made about a Bernoulli experiment , that is an experiment which tests the statistical significance of deviations from a theoretically expected distribution (the binomial distribution) of observations into 2 categories. It is a goodness of fit test.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000302</td>
<td>IAO:0000115</td>
<td>"one sample t-test is a kind of Student's t-test which evaluates if a given sample can be reasonably assumed to be taken from the population.</td>
</tr>
<tr>
<td>The test compares the sample statistic (m) to the population parameter (M).</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The one sample t-test is the small sample analog of the z test, which is suitable for large samples.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000303</td>
<td>IAO:0000115</td>
<td>"two sample t-test is a null hypothesis statistical test which is used to reject or accept the hypothesis of absence of difference between the means over 2 randomly sampled populations.</td>
</tr>
<tr>
<td>It uses a t-distribution for the test and assumes that the variables in the population are normally distributed and with equal variances.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000306</td>
<td>IAO:0000115</td>
<td>a polynomial contrast is a contrast which...@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000307</td>
<td>IAO:0000115</td>
<td>treatment contrast is a contrast which allows to test how linear model coefficients of categorical variables are interpreted in case where  the “first” level (aka, the baseline) is included into the intercept and all subsequent levels have a coefficient that represents their difference from the baseline.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000308</td>
<td>IAO:0000115</td>
<td>the sum contrast is a contrast in which each coefficient compares the corresponding level of the factor to the average of the other levels@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000311</td>
<td>IAO:0000115</td>
<td>"a central composite design is a study design which contains an imbedded factorial or fractional factorial design with center points that is augmented with a group of so-called 'star points' that allow estimation of curvature.</td>
</tr>
<tr>
<td>A CCD design with k factors has 2k star points.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000314</td>
<td>IAO:0000115</td>
<td>upper confidence limit is a data item which is a largest value bounding a confidence interval@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000315</td>
<td>IAO:0000115</td>
<td>lower confidence limit is a data item which is a lowest value bounding a confidence interval@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000316</td>
<td>IAO:0000115</td>
<td>root-mean-square standardized effect is a statistic which denotes effect size in the context of analysis of variance and corresponds to the square root of the arithmetic average of p  standardized effects  (effects normalized to be expressed in standard deviation units).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000318</td>
<td>IAO:0000115</td>
<td>omega-squared is a effect size estimate for variance explained which is less biased than the eta-squared coefficient.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000322</td>
<td>IAO:0000115</td>
<td>a contrast weight is a coefficient which multiplies a group mean, part of a linear combinaison defining a constrast as a weighted sum of group means, giving a 'weight' to a specific group mean hence the name.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000323</td>
<td>IAO:0000115</td>
<td>a contrast weight matrix is a information content entity which holds a set of contrast weight, coefficient used in a weighting sum of means defining a contrast@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000324</td>
<td>IAO:0000115</td>
<td>contrast weight estimate is a model parameter estimate which results from the computation from the data and that is used as input to a model fitting process@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000326</td>
<td>IAO:0000115</td>
<td>corrected Akaike information criteria is a modified version of the Akaike information criterion.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000333</td>
<td>IAO:0000115</td>
<td>kernel density estimation (KDE) is a non-parametric way to estimate the probability density function of a random variable@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000336</td>
<td>IAO:0000115</td>
<td>best linear unbiased prediction is a data transformation which predicts <TDB> under the assumption that the variable(s) under consideration have a random effect</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000337</td>
<td>IAO:0000115</td>
<td>breeding value estimation is a data transformation process aiming at computing breeding value estimates of an organism given a set of genomic (SNP) observations, pedigree information and/or phenotypic observations.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000338</td>
<td>IAO:0000115</td>
<td>breeding value estimation is a data transformation process aiming at computing breeding value estimates of an organism given a set of genomic (SNP) observations.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000339</td>
<td>IAO:0000115</td>
<td>breeding value estimation is a data transformation process aiming at computing breeding value estimates of an organism given a set of pedigree information.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000340</td>
<td>IAO:0000115</td>
<td>breeding value estimation is a data transformation process aiming at computing breeding value estimates of an organism given a set of phenotypic observations.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000342</td>
<td>IAO:0000115</td>
<td>genomic selection objective is a data transformation objective which is a special case of marker-assisted selection in which genetic markers covering the whole genome are used so that all quantitative trait loci (QTL) are in linkage disequilibrium with at least one marker.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000343</td>
<td>IAO:0000115</td>
<td>a dataset which is made up of genotypic information, that is presenting allele information at specific loci in a set of individuals of an organism.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000344</td>
<td>IAO:0000115</td>
<td>'has effect on' is a special case of the 'is about' relationship to be used for mixed effect models@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000345</td>
<td>IAO:0000115</td>
<td>'has fixed effect on' is a special case of the 'is about' relationship to be used with fixed effect models@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000346</td>
<td>IAO:0000115</td>
<td>a covariance structure is a data item which is part of a regression model and which indicates a pattern in the covariance matrix. The nature of covariance structure is specified before the regression analysis and various covariance structure may be tested and evaluated using information criteria to help choose the most suiteable model@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000349</td>
<td>IAO:0000115</td>
<td>spatial linear geometric anisotropic covariance structure is a type of covariance structure characterized by its anisotropy, i.e., the variation of properties can be different in directions x and y, which is this case give linear features.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000350</td>
<td>IAO:0000115</td>
<td>spatial spherical geometric anisotropic covariance structure is a type of covariance structure characterized by its anisotropy, i.e., the variation of properties can be different in directions x and y, which is this case give spherical features.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000351</td>
<td>IAO:0000115</td>
<td>spatial gaussian geometric anisotropic covariance structure is a type of covariance structure characterized by its anisotropy, i.e., the variation of properties can be different in directions x and y, which is this case give gaussian features.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000352</td>
<td>IAO:0000115</td>
<td>spatial exponential geometric anisotropic covariance structure is a type of covariance structure characterized by its anisotropy, i.e., the variation of properties can be different in directions x and y, which is this case give exponential features.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000353</td>
<td>IAO:0000115</td>
<td>spatial exponential anisotropic covariance structure is a type of covariance structure characterized by its anisotropy, i.e., the variation of properties can be different in directions x and y, which is this case give exponential features.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000354</td>
<td>IAO:0000115</td>
<td>the banded heterogeneous Toeplitz covariance structure is a type of coviance structure which is often used to  analyzed and intepret repeated measure design.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000358</td>
<td>IAO:0000115</td>
<td>a form of covariance structure used to provide analysis ground s in the context of repeated measures datasets (longitudinal, time series)@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000359</td>
<td>IAO:0000115</td>
<td>"factor-analytic structure is a covariance structure which is specified for q factors</td>
</tr>
<tr>
<td>equal diagonal factor-analytic covariance structure is a type of factor analytic covariance structure specified for q factors, which includes a diagonal component for repeated measures.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000360</td>
<td>IAO:0000115</td>
<td>no diagonal factor-analytic covariance structure is a type of factor analytic covariance structure specified for q factors, which does not include a diagonal component for repeated measures.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000361</td>
<td>IAO:0000115</td>
<td>factor-analytic structure is a type of heterogeneous covariance structure which is specified for q factors@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000362</td>
<td>IAO:0000115</td>
<td>compound symmetry covariance structure is a covariance structure which means that all the variances are equal and all the covariances are equal.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000363</td>
<td>IAO:0000115</td>
<td>heterogenous compound symmetry structure is a compound symmetry covariance structure which has a different variance parameter for each diagonal element, and it uses the square roots of these parameters in the off-diagonal entries.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000364</td>
<td>IAO:0000115</td>
<td>first order autoregressive moving average covariance structure is a type of covariance structure which is used in the context of time series analysis@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000365</td>
<td>IAO:0000115</td>
<td>first order autoregressive covariance structure is a covariance structure where correlations among errors decline exponentially with distance@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000369</td>
<td>IAO:0000115</td>
<td>repeated measure analysis is a kind of data transformation which deals with signals measured in the same experimental units at different times and, possibly, under different conditions over a period of time. Data produced by longitudinal studies qualify for such analysis. Since measurements are made on the same experimental units a number of times, they are likely to be correlated. Repeated measure analysis usually takes into consideration the possibility of correlation with time. It does so by specifying covariance structure in the analysis@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000370</td>
<td>IAO:0000115</td>
<td>the ordinary least squares estimation is a model parameter estimation for a linear regression model when the errors are uncorrelated and equal in variance. Is the Best Linear Unbiased Estimation (BLUE) method under these assumptions, Uniformly Minimum-Variance Unbiased Estimator (UMVUE) with addition of a Gaussian assumption.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000371</td>
<td>IAO:0000115</td>
<td>the weighted least squares estimation is a model parameter estimation for a linear regression model with errors that independent but have heterogeneous variance. Difficult to use use in practice, as weights must be set based on the variance which is usually unknown. If true variance is known, it is the Best Linear Unbiased Estimation (BLUE) method under these assumptions, Uniformly Minimum-Variance Unbiased Estimator (UMVUE) with addition of a Gaussian assumption.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000372</td>
<td>IAO:0000115</td>
<td>"the generalized least squares estimation is a model parameter estimation for a linear regression model with errors that are dependent and (possibly) have heterogeneous variance. Difficult to use use in practice, as covariance matrix of the errors must known to \""whiten\"" data and model. If true covariance is known, it is the Best Linear Unbiased Estimation (BLUE) method under these assumptions, Uniformly Minimum-Variance Unbiased Estimator (UMVUE) with addition of a Gaussian assumption.@en"</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000373</td>
<td>IAO:0000115</td>
<td>the iteratively reweighted least squares estimation is a model parameter estimation which is a practical implementation of Weighted Least Squares, where the heterogeneous variances of the errors are estimated from the residuals of the regression model, providing an estimate for the weights. Each successive estimate of the weights improves the estimation of the regression parameters, which in turn are used to compute residuals and update the weights@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000374</td>
<td>IAO:0000115</td>
<td>the feasible generalized least squares estimation is a model parameter estimation which is a practical implementation of Generalised Least Squares, where the covariance of the errors is estimated from the residuals of the regression model, providing the information needed to whiten the data and model. Each successive estimate of the whitening matrix improves the estimation of the regression parameters, which in turn are used to compute residuals and update the whitening matrix.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000375</td>
<td>IAO:0000115</td>
<td>a residual mean square is a data item which is obtained by dividing the sum of squared residuals (SSR) by the number of degrees of freedom</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000380</td>
<td>IAO:0000115</td>
<td>'has interaction effect on' is a special case of the 'is about' relationship to be used for mixed effect models@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000381</td>
<td>IAO:0000115</td>
<td>'has random effect on' is a special case of the 'is about' relationship to be used for random effect models@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000382</td>
<td>IAO:0000115</td>
<td>'has order in sequence' is a special case of the 'is about' relation being used to be able to enumerate the different terms within a linear mixed model formula (thus assinging and order to random effect terms, fixed effect terms, interaction effect terms and error terms).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000383</td>
<td>IAO:0000115</td>
<td>a data transformation that finds a contrast value (the contrast estimate) by computing the weighted sum of model parameter estimates using a set of contrast weights.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000384</td>
<td>IAO:0000115</td>
<td>estimate of a contrast obtained by computing the weighted sum of model parameter estimates using a set of contrast weights.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000385</td>
<td>IAO:0000115</td>
<td>an estimate of the standard deviation of a contrast estimate sampling distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000389</td>
<td>IAO:0000115</td>
<td>"a power-law probability distribution is a probability distribution whose density function (or mass function in the discrete case) has the form</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>p(x) = L(x) . x^{-alpha}</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>where alpha is a parameter &gt;1 and L(x) is a slowly varying function.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000391</td>
<td>IAO:0000115</td>
<td>an annotation property to provide a canonical command to invoke a method implementation using Python programming language@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000393</td>
<td>IAO:0000115</td>
<td>"the Pareto type-II probability distribution is a continuous probability distribution which is defined by a probability density function characterized by 2 parameters, alpha and lambda, 2 real, strictly positive numbers. alpha is known as the shape parameter while lambda is known as the scale parameter.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>the function defines the probably of a continous random variable according to the following:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>p(x) = {\alpha \over \lambda} \left[{1+ {x \over \lambda}}\right]^{-(\alpha+1)}, \qquad x \geq 0,@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000401</td>
<td>IAO:0000115</td>
<td>"the sample mean of sample of size n with n observations is an arithmetic mean computed over n number of observations on a statistical sample.</td>
</tr>
<tr>
<td>The sample mean, denoted x¯ and read “x-bar,” is simply the average of the n data points x1, x2, ..., xn:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>x¯=x1+x2+⋯+xnn=1n∑i=1nxi</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The sample mean summarizes the \""location\"" or \""center\"" of the data.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>the sample mean is a measure of dispersion of the observations made on the sample and provides an unbias estimate of the population mean@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000402</td>
<td>IAO:0000115</td>
<td>"the population mean or distribution mean is a parameter of a probability distribution or population indicative of the data dispersion. For continous probabibility distribution, the population mean is computed using the probability density function, for discrete probability distributions, a mass density function is used instead.</td>
</tr>
<tr>
<td>A population mean can be estimated by computing a sample mean@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000404</td>
<td>IAO:0000115</td>
<td>the most common series or system of written mathematical symbols used to represent the entity@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000409</td>
<td>IAO:0000115</td>
<td>the likelihood ratio is a ratio which is formed by dividing the post-test odds with the pre-test odds in the context of a Bayesian formulation@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000410</td>
<td>IAO:0000115</td>
<td>the likelihood ratio of negative results is a ratio which is formed  by dividing the difference between 1 and sensitivity of the test by the specificity value of a test. This can be expressed also as dividing the probability of a person who has the disease testing negative by the probability of a person who does not have the disease testing negative.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000411</td>
<td>IAO:0000115</td>
<td>the likelihood ratio of positive results is a ratio which is form by dividing the sensitivity value of a test by the difference between 1 and specificity of the test. This can be expressed also as dividing the probability of the test giving a positive result when testing an affected subject versus the probability of the test giving a positive result when a subject is not affected.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000414</td>
<td>IAO:0000115</td>
<td>mortality is a ratio  formed by the number of deaths due to a disease divided by the total population size.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000415</td>
<td>IAO:0000115</td>
<td>"in the context of binary classification, accuracy is defined as the proportion of true results (both true positives and true negatives) to the total number of cases examined (the sum of true positive, true negative, false positive and false negative).</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>It can be understood as a measure of the proximity of measurement results to the true value.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Accuracy is a metric used in the context of classification tasks to evaluate the proportion of correctly predicted instances among the total instances.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Key Points:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Use Case: Classification performance evaluation.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Metric: Measures the proportion of correct predictions.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Interpretation: Higher values indicate better classification performance.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000416</td>
<td>IAO:0000115</td>
<td>"precision or positive predictive value is defined as the proportion of the true positives against all the positive results (both true positives and false positives)</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A proportion in which the numerator represents the correctly detected items within the denominator that represents all items detected.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000418</td>
<td>IAO:0000115</td>
<td>a measure of heterogeneity in meta-analysis  is a data item which aims to describe the variation in study outcomes between studies.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000423</td>
<td>IAO:0000115</td>
<td>the proportion of individuals in a population with the outcome of interest@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000427</td>
<td>IAO:0000115</td>
<td>restricted maximum likelihood estimation is a kind of maximum likelihood estimation data transformation which estimates the variance components of random-effects in univariate and multivariate meta-analysis. in contrast to 'maximum likelihood estimation', reml can produce unbiased estimates of variance and covariance parameters.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000428</td>
<td>IAO:0000115</td>
<td>"maximum likelihood estimation (MLE) is a method of estimating the parameters of a statistical model, given observations. MLE attempts to find the parameter values that maximize the likelihood function, given the observations.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The method of maximum likelihood is based on the likelihood function, {\displaystyle {\mathcal {L}}(\theta \,;x)} {\displaystyle {\mathcal {L}}(\theta \,;x)}. We are given a statistical model, i.e. a family of distributions {\displaystyle {f(\cdot \,;\theta )\mid \theta \in \Theta }} {\displaystyle {f(\cdot \,;\theta )\mid \theta \in \Theta }}, where {\displaystyle \theta } \theta  denotes the (possibly multi-dimensional) parameter for the model. The method of maximum likelihood finds the values of the model parameter, {\displaystyle \theta } \theta , that maximize the likelihood function, {\displaystyle {\mathcal {L}}(\theta \,;x)} {\displaystyle {\mathcal {L}}(\theta \,;x)}. I@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000430</td>
<td>IAO:0000115</td>
<td>a random effect meta analysis procedure defined by Hartung and Knapp and by Sidik and Jonkman which performs better than DerSimonian and Laird approach, especially when there is heterogeneity and the number of studies in the meta-analysis is small.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000431</td>
<td>IAO:0000115</td>
<td>a meta analysis which relies on the computation of the DerSimonian and Leard estimator as a measure of heterogeneity over a set of studies.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000432</td>
<td>IAO:0000115</td>
<td>"a meta analysis which relies on the computation of the Hunter and Schmidt estimator as a measure of heterogeneity over a set of studies by considering the weighted mean of the raw correlation coefficient. Hunter and Schmidt developed what is commonly termed validity generalization procedures (Schmidt and Hunter, 1977). These involve correcting the effect sizes in the meta-analysis for sampling, and measurement error</td>
</tr>
<tr>
<td>and range restriction.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000435</td>
<td>IAO:0000115</td>
<td>a probability distribution scale parameter is a measure of variation which is set by the operator when selecting a parametric probability distribution and which defines how spread the distribution is. The larger  the value of the scale parameter is, the more spread out the distribution.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000436</td>
<td>IAO:0000115</td>
<td>a probability distribution shape parameter is a data item which is set by the operator when selecting a parametric probability distribution and which dictates the way the profile but not the location or size of the distribution plot looks like.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000437</td>
<td>IAO:0000115</td>
<td>a scale estimator is a measurement datum (a statistic) which is calculated to approach the actual scale parameter of a probability distribution from observed data.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000438</td>
<td>IAO:0000115</td>
<td>a log-normal (or lognormal) distribution is a continuous probability distribution of a random variable whose logarithm is normally distributed. Thus, if the random variable {\displaystyle X} X is log-normally distributed, then {\displaystyle Y=\ln(X)} Y=\ln(X) has a normal distribution. Likewise, if {\displaystyle Y} Y has a normal distribution, then {\displaystyle X=\exp(Y)} X=\exp(Y) has a log-normal distribution. A random variable which is log-normally distributed takes only positive real values. The distribution is occasionally referred to as the Galton distribution or Galton's distribution, after Francis Galton.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000439</td>
<td>IAO:0000115</td>
<td>outlier detection testing objective is a statistical objective of a data transformation which aims to test a null hypothesis that an observation is not an outlier.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000444</td>
<td>IAO:0000115</td>
<td>"a split-plot design is kind of factorial design  which is used when running a full factorial completely randomized design is inpractical, either for cost or practicalities (e.g. equipment, fields), in other words, when a restricted randomization has to be applied. A split-plot design is used whenever practioners fix the level of 'hard to change factor' and run all the combinations of the other factors. The hard to change factor is also refered to as the 'whole plot' factor, while the remainders of the factors are refered to as 'split plot factor'.</td>
</tr>
<tr>
<td>Performing a split-plot design therefore means fixing one factor level, and then applying the treatments formed by the cartesian products of the levels for the other factors. A mininum of 2 factors are required and one being applied before the other(s).@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000445</td>
<td>IAO:0000115</td>
<td>a split split plot design is a study design where restricted randomization affect 2 study factors (and not 1 as in split-plot design). Such design is only possible if at least 3 independent variables are present.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000447</td>
<td>IAO:0000115</td>
<td>"a 'whole plot number' is a data item used to count and identify the actual piece of land (in the case of real field based trials) used in a split plot design experiment and receiving treatments corresponding to the levels of a factor whose randomization is restricted (these factors are known as 'hard to change' factors).</td>
</tr>
<tr>
<td>In the case of non-field based trials, the 'whole plot' is a metaphor.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000448</td>
<td>IAO:0000115</td>
<td>"a 'sub plot number' is a data item used to count and identify the actual piece of land located within a 'whole plot', in the case of real field based trials using a split-plot design, and received  completely randomized treatments corresponding to the factor levels combinations of the remainder factors declared in the experiment.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>in the case of 'split-split plot design',  sub-plots also receive treatments corresponding to a factor whose randomization is restriction. In such configuration, each 'sub-plot' is itself divided into 'sub sub-plot', which then received the remainder of the treatments in completely randomized fashion.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>In the case of non-field based trials, the notion 'sub-plot' is a metaphor.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000449</td>
<td>IAO:0000115</td>
<td>"a 'sub sub-plot number' is a data item used to count and identify the actual piece of land located within a 'sub plot', in the case of real field based trials using a split-split-plot design, and received  completely randomized treatments corresponding to the factor levels combinations of the remainder factors declared in the experiment.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>in the case of 'split-split plot design',  sub-plots also receive treatments corresponding to a factor whose randomization is restriction. In such configuration, each 'sub-plot' is itself divided into 'sub sub-plot', which then received the remainder of the treatments in completely randomized fashion.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>In the case of non-field based trials, the notion 'sub sub-plot' is a metaphor.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000450</td>
<td>IAO:0000115</td>
<td>"\""Wilks'  lambda  is  a  test  statistic  used  in  multivariate  analysis  of  variance (MANOVA)  to  test  whether  there  are  differences  between  the  means  of identified  groups  of  subjects  on  a  combination  of  dependent  variables.\""@en"</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000452</td>
<td>IAO:0000115</td>
<td>"\""The Lawley–Hotelling trace is used to test the equality of mean vectors of k p‐variate normal distributions with common but unknown covariance matrix. The explicit form of the null distribution of T$_{0}^{2}$equation image is the F distribution. The asymptotic null distribution is the chi‐square distribution. The power function of the test is described and its power is compared with the likelihood ratio test. \""@en"</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000454</td>
<td>IAO:0000115</td>
<td>"\""The multivariate analysis of variance, or MANOVA, is a procedure for comparing multivariate sample means. As a multivariate procedure, it is used when there are two or more dependent variables, and is typically followed by significance tests involving individual dependent variables separately.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>It helps to answer:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>1. Do changes in the independent variable(s) have significant effects on the dependent variables?</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>2. What are the relationships among the dependent variables?</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>3. What are the relationships among the independent variables?\""@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000459</td>
<td>IAO:0000115</td>
<td>group sequential design is a study design used in clinical trial settings in which interim analyses of the data are conducted after groups of patients are recruited. After each interim analysis, the trial may stop early if the evidence so far shows the new treatment is particularly effective or ineffective. Such designs are ethical and cost-effective, and so are of great interest in practice.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000460</td>
<td>IAO:0000115</td>
<td>interim analysis is a data transformation used to analyzed studies implementing a group-sequential design, to evaluate and  interpret the accumulating information during a clinical trial. It means that the analysis of data that is conducted before full data collection has been completed. Clinical trials are unusual in that enrollment of patients is a continual process staggered in time. This means that if a treatment is particularly beneficial or harmful compared to the concurrent placebo group while the study is on-going, the investigators are ethically obliged to assess that difference using the data at hand and to make a deliberate consideration of terminating the study earlier than planned.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000461</td>
<td>IAO:0000115</td>
<td>"the O'brien-Flemming boundary analysis is a kind of interim-analysis method implemented by O'brien and Flemming to account for the</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>As all frequentist methods of the same type, it focuses on controlling the type I error rate as the repeated hypothesis testing of accumulating data increases the type I error rate of a clinical trial.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000467</td>
<td>IAO:0000115</td>
<td>the model random effect term is model term which aims to account for the unwanted variability in the data associated with a range of independent variables which are not the primary interest in the dataset. It is there also known as the variance component of the model</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000468</td>
<td>IAO:0000115</td>
<td>a model fixed effect term is a model term which accounts for  variation explained by an independent variable and its levels.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000469</td>
<td>IAO:0000115</td>
<td>a model interaction effect term is a model term which accounts for variation explained by the combined effects of the factor levels of more than one (usually 2) independent variables.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000470</td>
<td>IAO:0000115</td>
<td>a model error term is a model term which accounts for  residual variation not explained by the other components (fixed and random effect terms)</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000471</td>
<td>IAO:0000115</td>
<td>a estimate is a data item which is computed from a dataset to provide an approximated value (an estimator) for a 'statistical parameter'  (a 'characteristics/parameter' of the true underlying distribution) of a real population.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000475</td>
<td>IAO:0000115</td>
<td>a data transformation to determine the number of degree of freedom@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000478</td>
<td>IAO:0000115</td>
<td>a dataset which is made up of pedigree information, that is presenting ancestry or lineage information in a set of individuals of an organism.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000481</td>
<td>IAO:0000115</td>
<td>a data transformation which calculate predictions of breeding values using an animal model and a relationship matrix calculated from the genomic/genetic markers (G Matrix), in constrast to using Pedigree information as in BLUP, also known as ABLUP</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000482</td>
<td>IAO:0000115</td>
<td>a data transformation which calculate estimates of genomic estimated breeding values (GEBVs) on an animal or plant model utilizing trait-specific marker information.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000485</td>
<td>IAO:0000115</td>
<td>the estimated breeding value of an organism is a data item computed to estimate the true breeding value defined as genetic merit of an organism, half of which will be passed on to its progeny.  While the exact breeding value can not been known, for performance traits it is possible to make good estimates. These estimates are called Estimated Breeding Values (EBVs).  EBVs are expressed in the units of measurement for each particular trait. These estimates are output of various estimation methods which differ depending on the underlying assumptions (equal variance of marker effect, all markers contributing to the trait) , the mathemical methods used (bayesian or non-bayesians) and the genetic inheritance models being considered (additive, dominant, epistatic) selected by the analysts.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000487</td>
<td>IAO:0000115</td>
<td>an additive genetic model is a data item which refers to the contributions to the final phenotype from more than one gene, or from alleles of a single gene (in heterozygotes), that combine in such a way that the sum of their effects in unison is equal to the sum of their effects individually.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000488</td>
<td>IAO:0000115</td>
<td>an additive genetic model is a data item which refer to the contributions to the final phenotype from more than one gene, or from alleles of a single gene (in heterozygotes), that combine in such a way that the sum of their effects in unison is equal to the sum of their individual effects and their dominance effect (of alleles at a single locus).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000489</td>
<td>IAO:0000115</td>
<td>an additive genetic model is a data item which refer to the contributions to the final phenotype from more than one gene, or from alleles of a single gene (in heterozygotes), that combine in such a way that the sum of their effects in unison is equal to the sum of their individual effect, their additive dominant (effect (of alleles at a single locus) and their epistasic effect (of alleles at more different loci).@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000493</td>
<td>IAO:0000115</td>
<td>a genotype matrix is a kind of genomic relationship matrix in the rawest of form and which simply corresponds to a matrix of individuals genotype for a given set of markers or genomic positions. Columns are snps or markers, Rows are individuals. Each column/row cell contains a genotype expressed as, in the genome is diploid, as a pair of characters chosen from ATGC where the dominant variant is uppercased and the recessive variant is lower cased.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000494</td>
<td>IAO:0000115</td>
<td>the MAF matrix is a genomic relationship matrix which is obtained from the genotype matrix by counting the number of minor alleles at each locus@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000495</td>
<td>IAO:0000115</td>
<td>"the M matrix is a genomic relationship matrix which is obtained by subtracting 1 to every value of the MAF matrix (gene content matrix). The values of the M matrix are only -1, 0 or 1 and makes computation easier.</td>
</tr>
<tr>
<td>M = MAF-1"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000497</td>
<td>IAO:0000115</td>
<td>the Z-matrix is a genomic relationship matrix which is obtained by substracted the M matrix with the P matrix. It is also known as the incidence matrix for the markers.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000499</td>
<td>IAO:0000115</td>
<td>"augmented design is a kind of experimental design where the goal is to compare existing (control) treatments with new treatments that have an experimental constraint of \""limited replication\"".  To understand limited replication, consider about experiments that may only allow a single representation of the new treatment, this limitation may be many times due to the cost associated  with the experiment, limited resources, or limited number of new units that can be used in the experiment. In contrast, the existing treatments are referred as checks and are generally replicated multiple times.  With augmented design one can estimate the following:</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>a) Differences between checks and new treatments,</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>b) Differences among new treatments,</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>c) Differences among check treatments, and</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>d) Differences among new and check treatments combined."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000500</td>
<td>IAO:0000115</td>
<td>a probability distribution location parameter is a data item which is set by the operator when selecting a parametric probability distribution and which dictates the way the location but not the profile or size of the distribution plot looks like.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000501</td>
<td>IAO:0000115</td>
<td>"the Weibull probability distribution is continuous probabibility distribution which is used to model time to fail, time to repair and material strength in material science. In biomedicine, the Weibull probability is used to in determining 'hazard functions'.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The 'location parameter' of the Weibull probability distribution can be used to define a failure-free zone.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>If the quantity X is a \""time-to-failure\"", the Weibull distribution gives a distribution for which the failure rate is proportional to a power of time. The shape parameter, k, is that power plus one, and so this parameter can be interpreted directly as follows:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A value of {\displaystyle k&lt;1\,} {\displaystyle k&lt;1\,} indicates that the failure rate decreases over time. This happens if there is significant \""infant mortality\"", or defective items failing early and the failure rate decreasing over time as the defective items are weeded out of the population. In the context of the diffusion of innovations, this means negative word of mouth: the hazard function is a monotonically decreasing function of the proportion of adopters;</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A value of {\displaystyle k=1\,} {\displaystyle k=1\,} indicates that the failure rate is constant over time. This might suggest random external events are causing mortality, or failure. The Weibull distribution reduces to an exponential distribution;</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A value of {\displaystyle k&gt;1\,} {\displaystyle k&gt;1\,} indicates that the failure rate increases with time. This happens if there is an \""aging\"" process, or parts that are more likely to fail as time goes on. In the context of the diffusion of innovations, this means positive word of mouth: the hazard function is a monotonically increasing function of the proportion of adopters. The function is first concave, then convex with an inflexion point at {\displaystyle (e^{1/k}-1)/e^{1/k},k&gt;1\,} {\displaystyle (e^{1/k}-1)/e^{1/k},k&gt;1\,}.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000502</td>
<td>IAO:0000115</td>
<td>statistical sampling is a planned process which aims at assembling a population of observation units (samples) in as an unbiaised manner as possible in order to obtain or infer information about the actual population these samples have been drawn.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000504</td>
<td>IAO:0000115</td>
<td>line intercept sampling  is a sampling process by which an element in a spatial region is included in a sample if it is intersected by a line chosen by the operator.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000508</td>
<td>IAO:0000115</td>
<td>stratified sampling is a statistical sampling method which divides the population into homogenous subpopulations, which are then sampled using random or systematic sampling methods</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000509</td>
<td>IAO:0000115</td>
<td>systematic sampling is a process for collecting samples and assembling a statistical sample using a system or method (.e.g unequal probabilities, without replacement, fixed sample size), as opposed to a random sampling.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000517</td>
<td>IAO:0000115</td>
<td>complete randomization is a group randomization where experimental units are randomly assigned to the entire set of groups defined by the experimental treatments.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000519</td>
<td>IAO:0000115</td>
<td>last observation carried forward data imputation is a type of data imputation which uses a very simple, self explanatory method for substituted a missing value for an observation. It should be noted that this method gives a biased estimate of the treatment effect and underestimates the variability of the estimated result and should be used cautiously.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000520</td>
<td>IAO:0000115</td>
<td>regression data imputation is a type of data imputation where missing values are replaced with the value of a regression function coefficient.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000521</td>
<td>IAO:0000115</td>
<td>substitution by the mean data imputation is a type of data imputation where missing values are replaced with the value the variable mean.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000522</td>
<td>IAO:0000115</td>
<td>multivariate imputation with chained equations (MICE) is a type of data imputation which uses an algorithm devised by Stef van Buuren and Karin Groothuis-Oudshoorn</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000523</td>
<td>IAO:0000115</td>
<td>k-nearest neighbour imputation is a data imputation which uses the k-nearest neighbour algorithm to compute a substitution value for the missing values. For every observation to be imputed, it identifies ‘k’ closest observations based on the euclidean distance and computes the weighted average (weighted based on distance) of these ‘k’ obs.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000525</td>
<td>IAO:0000115</td>
<td>a covariance matrix is a square matrix that contains the variances and covariances associated with several variables. The diagonal elements of the matrix contain the variances of the variables and the off-diagonal elements contain the covariances between all possible pairs of variables.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000526</td>
<td>IAO:0000115</td>
<td>the numerator relationship matrix is the matrix of <em>expected</em> additive genetic relationships between individuals. This matrix was originally used by Henderson (Henderson, C.R. 1976. A simple method for computing the inverse of a numerator relationship matrix used in prediction of breeding values. Biometrics 32:69-83.) to account for covariances between random effects, and therefore to use information from relatives in estimation of breeding value. Among the properties of the NRM matrix (also known as the A matrix), it is symmetric, the diagonal value correspond to 1+ the inbreeding coefficient for an individual.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000529</td>
<td>IAO:0000115</td>
<td>a scaled t distribution is a kind of Student's t distribution which is shifted by 'mean' and scaled by standard deviation 'sd'.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000530</td>
<td>IAO:0000115</td>
<td>a Bayesian model is a statistical model  where inference is based on using Bayes theorem to obtain a posterior distribution for a quantity (or quantities) of interest for some model (such as parameter values) based on some prior distribution for the relevant unknown parameters and the likelihood from the model.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000531</td>
<td>IAO:0000115</td>
<td>a prior probability distribution is a probability distribution used as input to a Bayesian model to represent a priori knowledge about a model parameter. Along with the acquired/observed data, it is used to compute a posterior distribution according to the Bayes theorem.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000532</td>
<td>IAO:0000115</td>
<td>a posterior probability distribution is a probability distribution computed in a Bayesian model approach given a prior distribution and a set of events/observations.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000534</td>
<td>IAO:0000115</td>
<td>genetic inheritance model is a data item defining the assumption used by a breeding value estimation method to consider when running the calculations.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000535</td>
<td>IAO:0000115</td>
<td>sampling from a probability distribution is a data transformation which aims at obtaining a sequence of random samples from a probability distribution for which direct sampling is difficult.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000537</td>
<td>IAO:0000115</td>
<td>the Metropolis–Hastings algorithm is a Markov chain Monte Carlo (MCMC) method for obtaining a sequence of random samples from a probability distribution for which direct sampling is difficult.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000538</td>
<td>IAO:0000115</td>
<td>a continuous multivariate probability distribution is a continuous probability distribution which describes the possible values, and corresponding probabilities, of two or more (usually three or more) associated random variables.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000539</td>
<td>IAO:0000115</td>
<td>a discrete multivariate probability distribution is a discrete probability distribution which describes the possible values, and corresponding probabilities, of two or more (usually three or more) associated random variables.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000541</td>
<td>IAO:0000115</td>
<td>a state space model is a kind of statistical model which describes the probabilistic dependence between the latent state variable and the observed measurement. The state or the measurement can be either continuous or discrete. The term “state space” originated in 1960s in the area of control engineering (Kalman, 1960). SSM provides a general framework for analyzing deterministic and stochastic dynamical systems that are measured or observed through a stochastic process.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000542</td>
<td>IAO:0000115</td>
<td>genomic estimated breeding value (GEBV) is an estimated breeding value derived from information in an organism DNA (genotype). GEBV is calculated differently to conventional Estimated Breeding Values using advanced modeling technique to deal with high dimensionality data.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000549</td>
<td>IAO:0000115</td>
<td>random forest procedure is a type of data transformation used in classification and statistical learning using regression. The random forest procedure is a meta estimator that fits a number of classifying decision trees on various sub-samples of the dataset (it operates by constructing a multitude of decision trees at training time) and use averaging to improve the predictive accuracy and control over-fitting. The sub-sample size is always the same as the original input sample size but the samples are drawn with replacement if bootstrap=True (default). The random forest procedure outputs the class that is the mode of the classes (classification) or mean prediction (regression) of the individual trees.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000550</td>
<td>IAO:0000115</td>
<td>"log likelihood is a data item which corresponds to the natural logarithm of the likelihood.</td>
</tr>
<tr>
<td>log likelihood is a data item commonly used to provide a measure of accuracy of a model."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000554</td>
<td>IAO:0000115</td>
<td>number of cross-validation segments is a count which is used as input parameter in a cross validation procedure to evaluate a statistical model.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000555</td>
<td>IAO:0000115</td>
<td>number of predictive components is a count used as input to the principle component analysis (PCA)</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000556</td>
<td>IAO:0000115</td>
<td>number of orthogonal components is a count used as input to the orthogonal partial least square discriminant analysis (OPLS-DA)</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000557</td>
<td>IAO:0000115</td>
<td>computed_from is a relation between 2 information content entity denoting how one is derived from another on through the application of a data transformation or computation process.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000559</td>
<td>IAO:0000115</td>
<td>"the Wald test is statistical test which computes a Wald chi-squared test for 1 or more coefficients, given their variance-covariance matrix.</td>
</tr>
<tr>
<td>The Wald test (also called the Wald Chi-Squared Test) is a way to find out if explanatory variables in a model are significant. “Significant” means that they add something to the model; variables that add nothing can be deleted without affecting the model in any meaningful way"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000560</td>
<td>IAO:0000115</td>
<td>the Rao-Scott score is a statistic which is used to test the hypothesis that all coefficients associated with a particular regression term are zero (or have some other specified values). the LRT uses a linear combination of chi-squared distributions</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000561</td>
<td>IAO:0000115</td>
<td>"the frequency (i.e., the proportion) of possible confidence intervals that contain the true value of their corresponding parameter. In other words, if confidence intervals are constructed using a given confidence level in an infinite number of independent experiments, the proportion of those intervals that contain the true value of the parameter will match the confidence level.</td>
</tr>
<tr>
<td>A probability measure of the reliability of an inferential statistical test that has been applied to sample data and which is provided along with the confidence interval for the output statistic."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000565</td>
<td>IAO:0000115</td>
<td>"a regression coefficient is a measure of association that is used as the coefficient of an independent variable in a regression model, of the dependent variable, which is linear in its parameters.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A value of zero means no association. The sign (positive or negative) reflects the direction of association.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>a regression coefficient is a measure of association  generated by a type of data transformation called a regression, which aims to model a response variable by expression the predictor variables as part of a function where variable terms are modified by a number. A regression coefficient is one such number.@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000572</td>
<td>IAO:0000115</td>
<td>a version of PLS used for classification, where the input y-block are group labels (categorical variable) rather than a continuous variable@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000575</td>
<td>IAO:0000115</td>
<td>a data transformation which finds principal component  by applying non-linear iterative partial least squares algorithm</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000577</td>
<td>IAO:0000115</td>
<td>a partial least square regression applied when there is only one variable in Y (the matrix of response variables), or it is desirable to model and optimize separately the performance of each of the variables in Y. This case is usually referred to as PLS1 regression (J = 1).</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000578</td>
<td>IAO:0000115</td>
<td>a partial least square regression applied to a multivariate response variable.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000579</td>
<td>IAO:0000115</td>
<td>improved kernel PLS is a data transformation which implement a very fast kernel algorithm for updating PLS models in a recursive manner and for exponentially discounting past data.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000580</td>
<td>IAO:0000115</td>
<td>variable importance in projection is a measure computed as part of a partial least square regression  to accumulate the importance of each variable j being reflected by w from each component.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000581</td>
<td>IAO:0000115</td>
<td>"a data transformation which compute the singular-value decomposition of a rectangular matrix.</td>
</tr>
<tr>
<td>The singular-value decomposition is very general in the sense that it can be applied to any m × n matrix whereas eigenvalue decomposition can only be applied to certain classes of square matrices."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000582</td>
<td>IAO:0000115</td>
<td>best linear unbiased estimator</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000583</td>
<td>IAO:0000115</td>
<td>"a completely randomized design is a type of design of experiment where the observation unit receive treatments (independent variable level) entirely at random. In other words, the observations unit are randomly assigned to treatments.</td>
</tr>
<tr>
<td>Completely randomized designs differ from randomized complete block design and should not be confused as in the latter, a blocking variable is first use to assign experimental units to blocks. Then only, the members of each block are then randomly assigned to different treatment groups"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000584</td>
<td>IAO:0000115</td>
<td>"the Wald statistic is a statistic is used during a Wald test, a test of significance of the regression coefficient; it is based on the asymptotic normality property of maximum likelihood estimates, and is computed as:</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>W = b * 1/Var(b) * b</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>In this formula, b stands for the parameter estimates, and Var(b) stands for the asymptotic variance of the parameter estimates. The Wald statistic is tested against the Chi-square distribution in the Wald test."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000585</td>
<td>IAO:0000115</td>
<td>degree of freedom calculation is a data transformation which is part of a stastical test and which aims to determine or estimate the number of degrees of freedom in a system.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000586</td>
<td>IAO:0000115</td>
<td>a restricted randomized design is a kind of study design which uses randomization to allocate observation unit to treatment but where intuitively poor allocations of treatments to experimental units are avoided, while retaining the theoretical benefits of randomization. This is often the case when so-called 'hard to change' factors are used in an experimental design.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000587</td>
<td>IAO:0000115</td>
<td>"the percentage of variance is an output of principal component analysis (PCRA), which is obtained by forming the ratio of an eigen-value divided by the sum of all eigen-values. This produces a \""percentage of variance\"" for each eigen-vector."</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000588</td>
<td>IAO:0000115</td>
<td>the scaled identity covariance structure is a type of covariance structure which has constant variance. The assumption is that there is no correlation between any elements.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000590</td>
<td>IAO:0000115</td>
<td>"median of the ratios corrected count is kind of count produced during an  RNA-Seq data normalization procedure which corresponds to dividing  counts  by sample-specific size factors determined by median ratio of gene counts relative to geometric mean per gene.</td>
</tr>
<tr>
<td>It was first described by Anders and Huber in 2010 (https://doi.org/10.1186/gb-2010-11-10-r106)</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Recommended use:</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>-\""median of the ratios corrected count\"" is suited for Differential Expression analysis or between samples.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>-\""median of the ratios corrected count\"" is NOT suited for gene count comparisons within a sample."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000592</td>
<td>IAO:0000115</td>
<td>the Rand index is a ratio, related to the notion of accuracy (STATO_0000415), which is used to compare the similarity of two clustering outcomes.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000593</td>
<td>IAO:0000115</td>
<td>the adjusted Rand index is a measure which rescales the Rand index, taking into account that random chance will cause some objects to occupy the same clusters, so the Rand Index will never actually be zero.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000594</td>
<td>IAO:0000115</td>
<td>"a confusion matrix is a 2 by 2 contingency table used to evaluate the performance of a classifier, often a machine-learning classifier and  that allows visualization of the performance of an algorithm, typically a supervised learning one. It defines two dimensions (\""actual\"" and \""predicted\""), and identical sets of \""classes\"" in both dimensions (each combination of dimension and class is a variable in the contingency table).@en"</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000595</td>
<td>IAO:0000115</td>
<td>the number of true positive is a count which denotes how many elements are correctly classified as having a feature they are actually known to be having (e.g. carrier of a pathogen).</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000596</td>
<td>IAO:0000115</td>
<td>the number of false positive is a count which denotes how many elements known to be void of feature are wrongly classified as having it (e.g. being diagnosed with a disease when one is totally healthy)</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000597</td>
<td>IAO:0000115</td>
<td>the number of true negative is a count which denotes how many elements are correctly classified as void of a feature they are actually known to be missing (e.g. free of pathogen).</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000598</td>
<td>IAO:0000115</td>
<td>the number of false negative is a count which denotes how many elements known to be having a feature are wrongly classified as being devoided of it (e.g. being given an all clear while being actually infected and carrier of a pathogen)</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000599</td>
<td>IAO:0000115</td>
<td>a point estimate is a data item which provides a particular value evaluating a population parameter@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000600</td>
<td>IAO:0000115</td>
<td>an interval estimate is a data item corresponding to a range of values likely to contain the population parameter of interest@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000601</td>
<td>IAO:0000115</td>
<td>simultaneous multiple testing method is a multiple testing correction method which...</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000602</td>
<td>IAO:0000115</td>
<td>sequential multiple testing method is a multiple testing correction method which...</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000603</td>
<td>IAO:0000115</td>
<td>a sequential multiple correction procedure which does not maintain a constant false positive rate but allows it to grow controllably.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000604</td>
<td>IAO:0000115</td>
<td>a type of sequential multiple testing correction method</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000605</td>
<td>IAO:0000115</td>
<td>a type of sequential multiple testing correction</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000606</td>
<td>IAO:0000115</td>
<td>q is the basic statistic for the studentized range distribution, which is used for multiple comparison procedures, such as the single step procedure Tukey's range test, the Newman–Keuls method, and the Duncan's step down procedure, and establishing confidence intervals that are still valid after data snooping has occurred</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000607</td>
<td>IAO:0000115</td>
<td>a proportion is a ratio which corresponds to the fraction of the total presenting a particular feature</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000609</td>
<td>IAO:0000115</td>
<td>@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000610</td>
<td>IAO:0000115</td>
<td>measure of association is a statistic which quantitatively represents a relationship between two or more variables@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000611</td>
<td>IAO:0000115</td>
<td>"measure of correlation is a measure of association between ordinal or continuous variables.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A value of 0 means no association.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A positive value means a positive association (as one variable increases, the other variable increases).</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>A negative value means a negative association (as one variable increases, the other variable decreases).</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>For correlation coefficients, the possible values range from +1 (perfect positive association) to -1 (perfect negative association)@en"</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000621</td>
<td>IAO:0000115</td>
<td>diagnostic yield is a proportion in which the numerator represents the correctly detected items within the denominator that represents all items tested.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000622</td>
<td>IAO:0000115</td>
<td>ratio-based measure of association is a measure of association which relies on a quotient of 2 quantities to indicate the strength of the association.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000627</td>
<td>IAO:0000115</td>
<td>"odds correspond to a ratio in which the numerator represents the probability that an event will occur and the denominator represents the probability that an event will not occur.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>'Odds' and 'Odds ratio' are different terms. 'Odds' is a ratio of probabilities. 'Odds ratio' is a ratio of two different odds.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Odds are calculated as p / (1-p) where p is the probability of event occurrence.  When p = 0, the odds = 0. When p = 1, the odds may be expressed as not calculable or as \""odds against = 0\"".</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Odds may be expressed as p:(1-p). Odds may be expressed as p:q where q = 1-p. Odds may be expressed as a:b where a and b are multiples of p and (1-p). Examples of different expressions of the same odds include 3:2, 3/2, 0.6:0.4, 0.6/0.4, and 1.5.</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Odds may be expressed as \""odds for\"" or \""odds in favor\"" (e.g. 1:5 for a \""3\"" on a 6-sided die) or \""odds against\"" (e.g. 5:1 against a \""3\"" on a 6-sided die).</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>The term \""betting odds\"" used in gambling that involves financial amounts in the formulation is not an \""Odds\"" in the definition of the Scientific Evidence Code System."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000633</td>
<td>IAO:0000115</td>
<td>a cutoff is an information content entity that represents or sets the boundary at which something changes.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000642</td>
<td>IAO:0000115</td>
<td>a matrix is a rectangular array of numbers, which are called entries of the matrix.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000643</td>
<td>IAO:0000115</td>
<td>the sample variance is a variance computed over the actual observations made, which correspond to a sample drawn from a population in an experiment. the sample variance can be used to estimate the true variance of the underlying population/distribution.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000646</td>
<td>IAO:0000115</td>
<td>"the population variance is a variance of the true population from which a sample is derived.</td>
</tr>
<tr>
<td>the population variance describes the variability of a characteristic in the population."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000650</td>
<td>IAO:0000115</td>
<td>"sampling variance refers to the variability in the estimates of a population parameter that arises from random sampling.</td>
</tr>
<tr>
<td>sampling variance is a variance of the sampling distribution of a random variable and estimates the dispersion of sample estimates about their expected value in hypothetical repetitions of the sample."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000651</td>
<td>IAO:0000115</td>
<td>the output of a statistical sampling, a draw from a distribution or a population of physical or immaterial entities.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000652</td>
<td>IAO:0000115</td>
<td>calibration in statistics refers to the process of ensuring that the predicted probabilities or scores from a statistical model accurately reflect the true probabilities or outcomes observed in the data. It is an essential aspect of predictive modeling to ensure the reliability and interpretability of model predictions, where the goal is to estimate the likelihood of certain events or outcomes.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000654</td>
<td>IAO:0000115</td>
<td>the objective of a data transformation to test a null hypothesis of absence of difference within subject holds.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000655</td>
<td>IAO:0000115</td>
<td>calibration plot is a line graph which plots values resulting from predictions againts values obtained through observation.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000656</td>
<td>IAO:0000115</td>
<td>"the slope of a line graph is a data item denoting the rate of change between the two variables represented on the graph.</td>
</tr>
<tr>
<td>the slope is used to visualize and interpret the relationship between two variables."</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000657</td>
<td>IAO:0000115</td>
<td>an intercept is a data item which corresponds to where a graph line cuts (intercepts) an coordinates axis.</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000696</td>
<td>IAO:0000115</td>
<td>a value-time curve is a graph which plot time on the x-axis versus the value of a variable of interest as delivered by a process. It is used to represent the relationship there is the value and the time it takes to achieve that value. It is commonly used in project management or business analysis to evaluate and optimize how efficiently value is delivered over time.@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000697</td>
<td>IAO:0000115</td>
<td>an homogeneity test is a statistical test aiming at evaluate if the statisical measure from several random samples are similar@en</td>
</tr>
<tr>
<td>INFO</td>
<td>lowercase_definition</td>
<td>STATO:0000701</td>
<td>IAO:0000115</td>
<td>a statistical test which test for homogeneity of proportions, which is used when comparing proportions observed across multiple groups. It relies on frequencies calculated in contingency tables. It determines is proportions are consistent.</td>
</tr>
<tr>
<td>INFO</td>
<td>missing_superclass</td>
<td>BFO:0000001</td>
<td>rdfs:subClassOf</td>
<td></td>
</tr>
</tbody>
</table>







  
  






                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
    
    
      <a href="https://www.fiz-karlsruhe.de/en/bereiche/information-service-engineering" target="_blank">Information Service Engineering Group at FIZ Karlsruhe</a>
    
  </div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "../../..", "features": ["content.code.copy", "toc.follow", "navigation.top", "header.autohide"], "search": "../../../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
    
  </body>
</html>